<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="" />


<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Charlotte Soneson</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-pdf-o"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="resources.html">
    <span class="fa fa-cogs"></span>
     
    Resources
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-mortar-board"></span>
     
    Teaching
  </a>
</li>
<li>
  <a href="https://github.com/csoneson">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
<li>
  <a href="https://twitter.com/CSoneson">
    <span class="fa fa-twitter"></span>
     
    Twitter
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<div id="preprints-and-articles-awaiting-peer-review" class="section level2">
<h2>Preprints and articles awaiting peer review</h2>
<ul>
<li>Lukas M. Weber, Malgorzata Nowicka, <strong>Charlotte Soneson</strong>, Mark D. Robinson: diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering. bioRxiv doi:<a href="https://doi.org/10.1101/349738" target="_blank"> https://doi.org/10.1101/349738</a>. Posted March 29, 2019.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/349738" data-hide-no-mentions="true">

</div></li>
<li><strong>Charlotte Soneson</strong>, Yao Yao, Anna Bratus-Neuenschwander, Andrea Patrignani, Mark D. Robinson, Shobbir Hussain: A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes. bioRxiv doi:<a href="https://doi.org/10.1101/574525" target="_blank"> https://doi.org/10.1101/574525</a>. Posted March 11, 2019.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/574525" data-hide-no-mentions="true">

</div></li>
<li>Robert A. Amezquita, Vince J. Carey, Lindsay N. Carpp, Ludwig Geistlinger, Aaron T. L. Lun, Federico Marini, Kevin Rue-Albrecht, Davide Risso, <strong>Charlotte Soneson</strong>, Levi Waldron, Hervé Pagès, Mike Smith, Wolfgang Huber, Martin Morgan, Raphael Gottardo, Stephanie C. Hicks: Orchestrating Single-Cell Analysis with Bioconductor. bioRxiv doi:<a href="https://doi.org/10.1101/590562" target="_blank"> https://doi.org/10.1101/590562</a>. Posted March 27, 2019.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/590562" data-hide-no-mentions="true">

</div></li>
<li>Koen Van Den Berge, Katharina Hembach, <strong>Charlotte Soneson</strong>, Simone Tiberi, Lieven Clement, Michael I Love, Rob Patro, Mark Robinson: RNA sequencing data: hitchhiker’s guide to expression analysis. PeerJ doi:<a href="https://doi.org/10.7287/peerj.preprints.27283v1" target="_blank"> https://doi.org/10.7287/peerj.preprints.27283v1</a>. Posted November 24, 2018.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.7287/peerj.preprints.27283v1" data-hide-no-mentions="true">

</div></li>
<li>Lukas M. Weber, Wouter Saelens, Robrecht Cannoodt, <strong>Charlotte Soneson</strong>, Alexander Hapfelmeier, Paul P. Gardner, Anne-Laure Boulesteix, Yvan Saeys, Mark D. Robinson: Essential guidelines for computational method benchmarking. ArXiv ID:<a href="https://arxiv.org/abs/1812.00661" target="_blank"> https://arxiv.org/abs/1812.00661</a>. Posted Apr 14, 2019.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-arxiv-id="1812.00661" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section" class="section level2">
<h2>2019</h2>
<ul>
<li>Orjuela S, Huang R, Hembach KM, Robinson MD, <strong>Soneson C</strong>: ARMOR: An <em>A</em>utomated <em>R</em>eproducible <em>MO</em>dular Workflow for Preprocessing and Differential Analysis of <em>R</em>NA-seq Data. G3 (Bethesda) : (2019). DOI: 10.1534/g3.119.400185. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31088905" target="_blank"> PMID 31088905</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1534/g3.119.400185" data-hide-no-mentions="true">

</div></li>
<li>Cheng HW, Onder L, Novkovic M, <strong>Soneson C</strong>, Lütge M, Pikor N, Scandella E, Robinson MD, Miyazaki JI, Tersteegen A, Sorg U, Pfeffer K, Rülicke T, Hehlgans T, Ludewig B: Origin and differentiation trajectories of fibroblastic reticular cells in the splenic white pulp. Nat Commun 10(1):1739 (2019). DOI: 10.1038/s41467-019-09728-3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30988302" target="_blank"> PMID 30988302</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/s41467-019-09728-3" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Love MI, Patro R, Hussain S, Malhotra D, Robinson MD: A junction coverage compatibility score to quantify the reliability of transcript abundance estimates and annotation catalogs. Life Sci Alliance 2(1): (2019). DOI: 10.26508/lsa.201800175. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30655364" target="_blank"> PMID 30655364</a>. <a href="https://github.com/csoneson/annotation_problem_txabundance" target="_blank"> GitHub repo</a>. <a href="https://github.com/csoneson/jcc" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/378539v1" target="_blank"> Preprint</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.26508/lsa.201800175" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-1" class="section level2">
<h2>2018</h2>
<ul>
<li>Love MI, <strong>Soneson C</strong>, Patro R: Swimming downstream: statistical analysis of differential transcript usage following Salmon quantification. F1000Res 7:952 (2018). DOI: 10.12688/f1000research.15398.3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30356428" target="_blank"> PMID 30356428</a>. <a href="https://github.com/mikelove/swimdown" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.15398.3" data-hide-no-mentions="true">

</div></li>
<li>Duò A, Robinson MD, <strong>Soneson C</strong>: A systematic performance evaluation of clustering methods for single-cell RNA-seq data. F1000Res 7:1141 (2018). DOI: 10.12688/f1000research.15666.2. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30271584" target="_blank"> PMID 30271584</a>. <a href="https://github.com/markrobinsonuzh/scrnaseq_clustering_comparison" target="_blank"> GitHub repo</a>. <a href="http://bioconductor.org/packages/DuoClustering2018/" target="_blank"> Software/Resources</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.15666.2" data-hide-no-mentions="true">

</div></li>
<li>Perez-Shibayama C, Gil-Cruz C, Cheng HW, Onder L, Printz A, Mörbe U, Novkovic M, Li C, Lopez-Macias C, Buechler MB, Turley SJ, Mack M, <strong>Soneson C</strong>, Robinson MD, Scandella E, Gommerman J, Ludewig B: Fibroblastic reticular cells initiate immune responses in visceral adipose tissues and secure peritoneal immunity. Sci Immunol 3(26): (2018). DOI: 10.1126/sciimmunol.aar4539. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30097537" target="_blank"> PMID 30097537</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1126/sciimmunol.aar4539" data-hide-no-mentions="true">

</div></li>
<li>Rue-Albrecht K, Marini F, <strong>Soneson C</strong>, Lun ATL: iSEE: Interactive SummarizedExperiment Explorer. F1000Res 7:741 (2018). DOI: 10.12688/f1000research.14966.1. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30002819" target="_blank"> PMID 30002819</a>. <a href="https://github.com/ltla/isee2018" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/iSEE/" target="_blank"> Software/Resources</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.14966.1" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Robinson MD: Bias, robustness and scalability in single-cell differential expression analysis. Nat Methods 15(4):255-261 (2018). DOI: 10.1038/nmeth.4612. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29481549" target="_blank"> PMID 29481549</a>. <a href="https://github.com/csoneson/conquer_comparison" target="_blank"> GitHub repo</a>. <a href="http://imlspenticton.uzh.ch:3838/conquer/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/143289v1" target="_blank"> Preprint</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nmeth.4612" data-hide-no-mentions="true">

</div></li>
<li>Van den Berge K, Perraudeau F, <strong>Soneson C</strong>, Love MI, Risso D, Vert JP, Robinson MD, Dudoit S, Clement L: Observation weights unlock bulk RNA-seq tools for zero inflation and single-cell applications. Genome Biol 19(1):24 (2018). DOI: 10.1186/s13059-018-1406-4. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29478411" target="_blank"> PMID 29478411</a>. <a href="https://github.com/statOmics/zinbwaveZinger" target="_blank"> GitHub repo</a>. <a href="https://www.biorxiv.org/content/10.1101/250126v1" target="_blank"> Preprint</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-018-1406-4" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Robinson MD: Towards unified quality verification of synthetic count data with countsimQC. Bioinformatics 34(4):691-692 (2018). DOI: 10.1093/bioinformatics/btx631. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29028961" target="_blank"> PMID 29028961</a>. <a href="https://github.com/csoneson/countsimQC" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/countsimQC/" target="_blank"> Software/Resources</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/bioinformatics/btx631" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-2" class="section level2">
<h2>2017</h2>
<ul>
<li>Van den Berge K, <strong>Soneson C</strong>, Robinson MD, Clement L: stageR: a general stage-wise method for controlling the gene-level false discovery rate in differential expression and differential transcript usage. Genome Biol 18(1):151 (2017). DOI: 10.1186/s13059-017-1277-0. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28784146" target="_blank"> PMID 28784146</a>. <a href="https://bioconductor.org/packages/stageR/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/109082v1" target="_blank"> Preprint</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-017-1277-0" data-hide-no-mentions="true">

</div></li>
<li>Gille D, Zangger N, <strong>Soneson C</strong>, Bütikofer U, Delorenzi M, Schwander F, Kopf-Bolanz KA, Chollet M, Walther B, Laederach K, Vergères G: Caloric dose-responsive genes in blood cells differentiate the metabolic status of obese men. J Nutr Biochem 43:156-165 (2017). DOI: 10.1016/j.jnutbio.2017.02.012. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28319853" target="_blank"> PMID 28319853</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.jnutbio.2017.02.012" data-hide-no-mentions="true">

</div></li>
<li>Neubert NJ, Tillé L, Barras D, <strong>Soneson C</strong>, Baumgaertner P, Rimoldi D, Gfeller D, Delorenzi M, Fuertes Marraco SA, Speiser DE: Broad and Conserved Immune Regulation by Genetically Heterogeneous Melanoma Cells. Cancer Res 77(7):1623-1636 (2017). DOI: 10.1158/0008-5472.CAN-16-2680. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28104684" target="_blank"> PMID 28104684</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1158/0008-5472.CAN-16-2680" data-hide-no-mentions="true">

</div></li>
<li>Sauerwald J, <strong>Soneson C</strong>, Robinson MD, Luschnig S: Faithful mRNA splicing depends on the Prp19 complex subunit <em>faint sausage</em> and is required for tracheal branching morphogenesis in <em>Drosophila</em>. Development 144(4):657-663 (2017). DOI: 10.1242/dev.144535. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087625" target="_blank"> PMID 28087625</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1242/dev.144535" data-hide-no-mentions="true">

</div></li>
<li>Zanini IM, <strong>Soneson C</strong>, Lorenzi LE, Azzalin CM: Human cactin interacts with DHX8 and SRRM2 to assure efficient pre-mRNA splicing and sister chromatid cohesion. J Cell Sci 130(4):767-778 (2017). DOI: 10.1242/jcs.194068. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28062851" target="_blank"> PMID 28062851</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1242/jcs.194068" data-hide-no-mentions="true">

</div></li>
<li>Mathias A, Perriard G, Canales M, <strong>Soneson C</strong>, Delorenzi M, Schluep M, Du Pasquier RA: Increased ex vivo antigen presentation profile of B cells in multiple sclerosis. Mult Scler 23(6):802-809 (2017). DOI: 10.1177/1352458516664210. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27503907" target="_blank"> PMID 27503907</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1177/1352458516664210" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-3" class="section level2">
<h2>2016</h2>
<ul>
<li>Neubert NJ, <strong>Soneson C</strong>, Barras D, Baumgaertner P, Rimoldi D, Delorenzi M, Fuertes Marraco SA, Speiser DE: A Well-Controlled Experimental System to Study Interactions of Cytotoxic T Lymphocytes with Tumor Cells. Front Immunol 7:326 (2016). DOI: 10.3389/fimmu.2016.00326. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27625650" target="_blank"> PMID 27625650</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.3389/fimmu.2016.00326" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Robinson MD: iCOBRA: open, reproducible, standardized and live method benchmarking. Nat Methods 13(4):283 (2016). DOI: 10.1038/nmeth.3805. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27027585" target="_blank"> PMID 27027585</a>. <a href="http://bioconductor.org/packages/release/bioc/html/iCOBRA.html" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/033431v1" target="_blank"> Preprint</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nmeth.3805" data-hide-no-mentions="true">

</div></li>
<li>Scholz G, Jandus C, Zhang L, Grandclément C, Lopez-Mejia IC, <strong>Soneson C</strong>, Delorenzi M, Fajas L, Held W, Dormond O, Romero P: Modulation of mTOR Signalling Triggers the Formation of Stem Cell-like Memory T Cells. EBioMedicine 4:50-61 (2016). DOI: 10.1016/j.ebiom.2016.01.019. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26981571" target="_blank"> PMID 26981571</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.ebiom.2016.01.019" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Matthes KL, Nowicka M, Law CW, Robinson MD: Isoform prefiltering improves performance of count-based methods for analysis of differential transcript usage. Genome Biol 17:12 (2016). DOI: 10.1186/s13059-015-0862-3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813113" target="_blank"> PMID 26813113</a>. <a href="https://github.com/markrobinsonuzh/diff_splice_paper" target="_blank"> GitHub repo</a>. <a href="https://www.biorxiv.org/content/10.1101/025387v2" target="_blank"> Preprint</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-015-0862-3" data-hide-no-mentions="true">

</div></li>
<li>Kuhle J, Barro C, Disanto G, Mathias A, <strong>Soneson C</strong>, Bonnier G, Yaldizli Ö, Regeniter A, Derfuss T, Canales M, Schluep M, Du Pasquier R, Krueger G, Granziera C: Serum neurofilament light chain in early relapsing remitting MS is increased and correlates with CSF levels and with MRI measures of disease severity. Mult Scler 22(12):1550-1559 (2016). DOI: 10.1177/1352458515623365. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26754800" target="_blank"> PMID 26754800</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1177/1352458515623365" data-hide-no-mentions="true">

</div></li>
<li>Schell MJ, Yang M, Missiaglia E, Delorenzi M, <strong>Soneson C</strong>, Yue B, Nebozhyn MV, Loboda A, Bloom G, Yeatman TJ: A Composite Gene Expression Signature Optimizes Prediction of Colorectal Cancer Metastasis and Outcome. Clin Cancer Res 22(3):734-45 (2016). DOI: 10.1158/1078-0432.CCR-15-0143. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26446941" target="_blank"> PMID 26446941</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1158/1078-0432.CCR-15-0143" data-hide-no-mentions="true">

</div></li>
<li>Schiffthaler B, Kostadima M, NGS Trainer Consortium., Delhomme N, Rustici G: Training in High-Throughput Sequencing: Common Guidelines to Enable Material Sharing, Dissemination, and Reusability. PLoS Comput Biol 12(6):e1004937 (2016). DOI: 10.1371/journal.pcbi.1004937. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27309738" target="_blank"> PMID 27309738</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1371/journal.pcbi.1004937" data-hide-no-mentions="true">

</div></li>
<li>SIB Swiss Institute of Bioinformatics Members.: The SIB Swiss Institute of Bioinformatics’ resources: focus on curated databases. Nucleic Acids Res 44(D1):D27-37 (2016). DOI: 10.1093/nar/gkv1310. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615188" target="_blank"> PMID 26615188</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/nar/gkv1310" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-4" class="section level2">
<h2>2015</h2>
<ul>
<li><strong>Soneson C</strong>, Love MI, Robinson MD: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Res 4:1521 (2015). DOI: 10.12688/f1000research.7563.2. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26925227" target="_blank"> PMID 26925227</a>. <a href="https://bioconductor.org/packages/tximport/" target="_blank"> Software/Resources</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.7563.2" data-hide-no-mentions="true">

</div></li>
<li>Fuertes Marraco SA, <strong>Soneson C</strong>, Delorenzi M, Speiser DE: Genome-wide RNA profiling of long-lasting stem cell-like memory CD8 T cells induced by Yellow Fever vaccination in humans. Genom Data 5:297-301 (2015). DOI: 10.1016/j.gdata.2015.06.024. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26484272" target="_blank"> PMID 26484272</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.gdata.2015.06.024" data-hide-no-mentions="true">

</div></li>
<li>Guinney J, Dienstmann R, Wang X, de Reyniès A, Schlicker A, <strong>Soneson C</strong>, Marisa L, Roepman P, Nyamundanda G, Angelino P, Bot BM, Morris JS, Simon IM, Gerster S, Fessler E, De Sousa E Melo F, Missiaglia E, Ramay H, Barras D, Homicsko K, Maru D, Manyam GC, Broom B, Boige V, Perez-Villamil B, Laderas T, Salazar R, Gray JW, Hanahan D, Tabernero J, Bernards R, Friend SH, Laurent-Puig P, Medema JP, Sadanandam A, Wessels L, Delorenzi M, Kopetz S, Vermeulen L, Tejpar S: The consensus molecular subtypes of colorectal cancer. Nat Med 21(11):1350-6 (2015). DOI: 10.1038/nm.3967. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26457759" target="_blank"> PMID 26457759</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nm.3967" data-hide-no-mentions="true">

</div></li>
<li>Johnsson K, <strong>Soneson C</strong>, Fontes M: Low Bias Local Intrinsic Dimension Estimation from Expected Simplex Skewness. IEEE Trans Pattern Anal Mach Intell 37(1):196-202 (2015). DOI: 10.1109/TPAMI.2014.2343220. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26353219" target="_blank"> PMID 26353219</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1109/TPAMI.2014.2343220" data-hide-no-mentions="true">

</div></li>
<li>Shchepachev V, Wischnewski H, <strong>Soneson C</strong>, Arnold AW, Azzalin CM: Human Mpn1 promotes post-transcriptional processing and stability of U6atac. FEBS Lett 589(18):2417-23 (2015). DOI: 10.1016/j.febslet.2015.06.046. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26213367" target="_blank"> PMID 26213367</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.febslet.2015.06.046" data-hide-no-mentions="true">

</div></li>
<li>Fuertes Marraco SA, <strong>Soneson C</strong>, Cagnon L, Gannon PO, Allard M, Abed Maillard S, Montandon N, Rufer N, Waldvogel S, Delorenzi M, Speiser DE: Long-lasting stem cell-like memory CD8+ T cells with a naïve-like profile upon yellow fever vaccination. Sci Transl Med 7(282):282ra48 (2015). DOI: 10.1126/scitranslmed.aaa3700. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25855494" target="_blank"> PMID 25855494</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1126/scitranslmed.aaa3700" data-hide-no-mentions="true">

</div></li>
<li>Lorenzi LE, Bah A, Wischnewski H, Shchepachev V, <strong>Soneson C</strong>, Santagostino M, Azzalin CM: Fission yeast Cactin restricts telomere transcription and elongation by controlling Rap1 levels. EMBO J 34(1):115-29 (2015). DOI: 10.15252/embj.201489559. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25398909" target="_blank"> PMID 25398909</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.15252/embj.201489559" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-5" class="section level2">
<h2>2014</h2>
<ul>
<li><strong>Soneson C</strong>, Gerster S, Delorenzi M: Batch effect confounding leads to strong bias in performance estimates obtained by cross-validation. PLoS One 9(6):e100335 (2014). DOI: 10.1371/journal.pone.0100335. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24967636" target="_blank"> PMID 24967636</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1371/journal.pone.0100335" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>: compcodeR–an R package for benchmarking differential expression methods for RNA-seq data. Bioinformatics 30(17):2517-8 (2014). DOI: 10.1093/bioinformatics/btu324. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24813215" target="_blank"> PMID 24813215</a>. <a href="https://bioconductor.org/packages/compcodeR/" target="_blank"> Software/Resources</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/bioinformatics/btu324" data-hide-no-mentions="true">

</div></li>
<li>Sun Y, Zhang W, Chen D, Lv Y, Zheng J, Lilljebjörn H, Ran L, Bao Z, <strong>Soneson C</strong>, Sjögren HO, Salford LG, Ji J, French PJ, Fioretos T, Jiang T, Fan X: A glioma classification scheme based on coexpression modules of EGFR and PDGFRA. Proc Natl Acad Sci U S A 111(9):3538-43 (2014). DOI: 10.1073/pnas.1313814111. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24550449" target="_blank"> PMID 24550449</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1073/pnas.1313814111" data-hide-no-mentions="true">

</div></li>
<li>Duffy D, Mamdouh R, Laird M, <strong>Soneson C</strong>, Le Fouler L, El-Daly M, Casrouge A, Decalf J, Abbas A, Eldin NS, Fontes M, Abdel-Hamid M, Mohamed MK, Rafik M, Fontanet A, Albert ML: The ABCs of viral hepatitis that define biomarker signatures of acute viral hepatitis. Hepatology 59(4):1273-82 (2014). DOI: 10.1002/hep.26901. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24500966" target="_blank"> PMID 24500966</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1002/hep.26901" data-hide-no-mentions="true">

</div></li>
<li>Missiaglia E, Jacobs B, D’Ario G, Di Narzo AF, <strong>Soneson C</strong>, Budinska E, Popovici V, Vecchione L, Gerster S, Yan P, Roth AD, Klingbiel D, Bosman FT, Delorenzi M, Tejpar S: Distal and proximal colon cancers differ in terms of molecular, pathological, and clinical features. Ann Oncol 25(10):1995-2001 (2014). DOI: 10.1093/annonc/mdu275. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25057166" target="_blank"> PMID 25057166</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/annonc/mdu275" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Fontes M: Incorporation of gene exchangeabilities improves the reproducibility of gene set rankings. Computational Statistics &amp; Data Analysis 71:588-598 (2014). DOI: 10.1016/j.csda.2012.07.026. .
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.csda.2012.07.026" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-6" class="section level2">
<h2>2013</h2>
<ul>
<li><strong>Soneson C</strong>, Delorenzi M: A comparison of methods for differential expression analysis of RNA-seq data. BMC Bioinformatics 14:91 (2013). DOI: 10.1186/1471-2105-14-91. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23497356" target="_blank"> PMID 23497356</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2105-14-91" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Torstensson A: Manliga och kvinnliga teknologers självbilder och deras stereotypbilder av teknologer. Högre Utbildning 3(1):3-14 (2013). DOI: . .
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-7" class="section level2">
<h2>2012</h2>
<ul>
<li>Shchepachev V, Wischnewski H, Missiaglia E, <strong>Soneson C</strong>, Azzalin CM: Mpn1, mutated in poikiloderma with neutropenia protein 1, is a conserved 3’-to-5’ RNA exonuclease processing U6 small nuclear RNA. Cell Rep 2(4):855-65 (2012). DOI: 10.1016/j.celrep.2012.08.031. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23022480" target="_blank"> PMID 23022480</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.celrep.2012.08.031" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Fontes M: A framework for list representation, enabling list stabilization through incorporation of gene exchangeabilities. Biostatistics 13(1):129-41 (2012). DOI: 10.1093/biostatistics/kxr023. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21908866" target="_blank"> PMID 21908866</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/biostatistics/kxr023" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-8" class="section level2">
<h2>2011</h2>
<ul>
<li>Fontes M, <strong>Soneson C</strong>: The projection score–an evaluation criterion for variable subset selection in PCA visualization. BMC Bioinformatics 12:307 (2011). DOI: 10.1186/1471-2105-12-307. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21798031" target="_blank"> PMID 21798031</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2105-12-307" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Fontes M: A method for visual identification of small sample subgroups and potential biomarkers. Annals of Applied Statistics 5(3):2131-2149 (2011). DOI: 10.1214/11-AOAS460. .
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1214/11-AOAS460" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-9" class="section level2">
<h2>2010</h2>
<ul>
<li><strong>Soneson C</strong>, Fontes M, Zhou Y, Denisov V, Paulsen JS, Kirik D, Petersén A, Huntington Study Group PREDICT-HD investigators.: Early changes in the hypothalamic region in prodromal Huntington disease revealed by MRI analysis. Neurobiol Dis 40(3):531-43 (2010). DOI: 10.1016/j.nbd.2010.07.013. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20682340" target="_blank"> PMID 20682340</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.nbd.2010.07.013" data-hide-no-mentions="true">

</div></li>
<li>Lilljebjörn H, <strong>Soneson C</strong>, Andersson A, Heldrup J, Behrendtz M, Kawamata N, Ogawa S, Koeffler HP, Mitelman F, Johansson B, Fontes M, Fioretos T: The correlation pattern of acquired copy number changes in 164 ETV6/RUNX1-positive childhood acute lymphoblastic leukemias. Hum Mol Genet 19(16):3150-8 (2010). DOI: 10.1093/hmg/ddq224. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20513752" target="_blank"> PMID 20513752</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/hmg/ddq224" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Lilljebjörn H, Fioretos T, Fontes M: Integrative analysis of gene expression and copy number alterations using canonical correlation analysis. BMC Bioinformatics 11:191 (2010). DOI: 10.1186/1471-2105-11-191. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20398334" target="_blank"> PMID 20398334</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2105-11-191" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-10" class="section level2">
<h2>2009</h2>
<ul>
<li>Björklund T, Hall H, Breysse N, <strong>Soneson C</strong>, Carlsson T, Mandel RJ, Carta M, Kirik D: Optimization of continuous in vivo DOPA production and studies on ectopic DA synthesis using rAAV5 vectors in Parkinsonian rats. J Neurochem 111(2):355-67 (2009). DOI: 10.1111/j.1471-4159.2009.06340.x. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19686242" target="_blank"> PMID 19686242</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1111/j.1471-4159.2009.06340.x" data-hide-no-mentions="true">

</div></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
