---
title: "Publications"
author: ""
output: html_document
editor_options: 
  chunk_output_type: console
---

```{r, echo = FALSE, message = FALSE, warning = FALSE}
## Google Scholar ID
id <- "c28GqkIAAAAJ"

## PubMed IDs to remove
pmid_remove <- c()

## PubMed IDs to add
pmid_add <- c("27309738", "26615188", "25057166")

## bioRxiv dois
biorxiv_dois <- c("10.1101/574525", "10.1101/590562", "10.1101/657874", 
                  "10.1101/713412")

## PeerJ dois
peerj_dois <- c()#c("10.7287/peerj.preprints.27283v2")

## ArXiv dois
arxiv_dois <- c()

## github repos
pmid_github <- rbind(
  c(pmid = "30655364", github = "https://github.com/csoneson/annotation_problem_txabundance"),
  c(pmid = "30356428", github = "https://github.com/mikelove/swimdown"),
  c(pmid = "30271584", github = "https://github.com/markrobinsonuzh/scrnaseq_clustering_comparison"),
  c(pmid = "30002819", github = "https://github.com/ltla/isee2018"),
  c(pmid = "29481549", github = "https://github.com/csoneson/conquer_comparison"),
  c(pmid = "29478411", github = "https://github.com/statOmics/zinbwaveZinger"),
  c(pmid = "29028961", github = "https://github.com/csoneson/countsimQC"),
  c(pmid = "26813113", github = "https://github.com/markrobinsonuzh/diff_splice_paper")
)
biorxiv_github <- rbind(
  c(doi = "", github = "")
)

## preprint links
pmid_preprints <- rbind(
  c(pmid = "30655364", preprint = "https://www.biorxiv.org/content/10.1101/378539v1"),
  c(pmid = "29481549", preprint = "https://www.biorxiv.org/content/10.1101/143289v1"),
  c(pmid = "29478411", preprint = "https://www.biorxiv.org/content/10.1101/250126v1"),
  c(pmid = "28784146", preprint = "https://www.biorxiv.org/content/10.1101/109082v1"),
  c(pmid = "27027585", preprint = "https://www.biorxiv.org/content/10.1101/033431v1"),
  c(pmid = "26813113", preprint = "https://www.biorxiv.org/content/10.1101/025387v2"),
  c(pmid = "31088905", preprint = "https://www.biorxiv.org/content/10.1101/575951v1"),
  c(pmid = "31098416", preprint = "https://www.biorxiv.org/content/10.1101/349738v3"),
  c(pmid = "31221194", preprint = "https://arxiv.org/abs/1812.00661")
)

## software packages/other resources
pmid_software <- rbind(
  c(pmid = "27027585", software = "http://bioconductor.org/packages/release/bioc/html/iCOBRA.html"),
  c(pmid = "30655364", software = "https://github.com/csoneson/jcc"),
  c(pmid = "30271584", software = "http://bioconductor.org/packages/DuoClustering2018/"),
  c(pmid = "30002819", software = "https://bioconductor.org/packages/iSEE/"),
  c(pmid = "29481549", software = "http://imlspenticton.uzh.ch:3838/conquer/"),
  c(pmid = "29028961", software = "https://bioconductor.org/packages/countsimQC/"),
  c(pmid = "28784146", software = "https://bioconductor.org/packages/stageR/"),
  c(pmid = "26925227", software = "https://bioconductor.org/packages/tximport/"),
  c(pmid = "24813215", software = "https://bioconductor.org/packages/compcodeR/")
)
biorxiv_software <- rbind(
  c(doi = "", software = "")
)

## names to highlight in publication list
highlight <- c("Soneson C", "Charlotte Soneson")

## publications not in PubMed/bioRxiv
unlisted_publications <- data.frame(
  title = c("Incorporation of gene exchangeabilities improves the reproducibility of gene set rankings.",
            "Manliga och kvinnliga teknologers självbilder och deras stereotypbilder av teknologer.",
            "A method for visual identification of small sample subgroups and potential biomarkers.", 
            "RNA Sequencing Data: Hitchhiker's Guide to Expression Analysis."),
  pubdate = c("", "", "", ""),
  pubyear = c("2014", "2013", "2011", "2019"),
  journal = c("Computational Statistics & Data Analysis", "Högre Utbildning", 
              "Annals of Applied Statistics", "Annual Review of Biomedical Data Science"),
  authors = c("Soneson C, Fontes M", "Soneson C, Torstensson A", "Soneson C, Fontes M", 
              "Van den Berge K, Hembach KM, Soneson C, Tiberi S, Clement L, Love MI, Patro R, Robinson MD"),
  volume = c("71", "3", "5", "2"),
  issue = c("", "1", "3", ""),
  pages = c("588-598", "3-14", "2131-2149", "139-173"),
  doi = c("10.1016/j.csda.2012.07.026", "", "10.1214/11-AOAS460",
          "10.1146/annurev-biodatasci-072018-021255"),
  pmid = c("", "", "", ""),
  elocationid = c("doi: 10.1016/j.csda.2012.07.026", "", "doi: 10.1214/11-AOAS460", 
                  "doi: 10.1146/annurev-biodatasci-072018-021255")
)

## F1000Research articles awaiting peer review
# f1000_preprints <- data.frame(
#   title = c("Swimming downstream: statistical analysis of differential transcript usage following Salmon quantification"),
#   pubdate = c("Posted June 27, 2018."),
#   pubyear = c("Preprints"),
#   journal = c("F1000Research"),
#   authors = c("Michael I Love, Charlotte Soneson, Rob Patro"),
#   volume = c(7),
#   issue = c(952),
#   pages = c(""),
#   doi = c("10.12688/f1000research.15398.1"),
#   pmid = c(""),
#   elocationid = c("")
# )
f1000_preprints <- NULL
```


```{r, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
suppressPackageStartupMessages({
  library(dplyr)
  library(rentrez)
  library(rvest)
  library(scholar)
})
```

```{r}
fix_null <- function(x) {
  if (is.null(x) || length(x) == 0) NA
  else x
}

make_link_button <- function(address, title) {
  sprintf('<a href="%s" target="_blank" class="btn btn-primary">%s</a>', 
                   address, title)
}

make_link_text <- function(address, title) {
  sprintf('<a href="%s" target="_blank"> %s</a>', address, title)
}

make_link_pmid <- function(pmid) {
  if (pmid != "") {
      sprintf('<a href="%s" target="_blank"> %s</a>',
          paste0("http://www.ncbi.nlm.nih.gov/pubmed/", pmid),
          paste0("PMID ", pmid))
  } else {
    ""
  }
}

make_link_altmetric <- function(doi) {
  sprintf('<div data-badge-popover="right" data-badge-type="4" data-doi="%s" data-hide-no-mentions="true" class="altmetric-embed"></div>',
          doi)
}

make_link_altmetric_arxiv <- function(arxivid) {
  sprintf('<div data-badge-popover="right" data-badge-type="4" data-arxiv-id="%s" data-hide-no-mentions="true" class="altmetric-embed"></div>',
          arxivid)
}
```


```{r}
## Get Google Scholar publication list
scholarpubs <- get_publications(id)

## Search PubMed
x <- entrez_search(db = "pubmed", term = "Soneson Charlotte[au]", retmax = 1000)

## Add and remove PubMed IDs manually
x$ids <- unique(c(base::setdiff(x$ids, pmid_remove), pmid_add))

## Extract info
summ <- entrez_summary(db = "pubmed", id = x$ids)
summ <- lapply(summ, function(w) {
  data.frame(title = fix_null(w$title), 
             pubdate = fix_null(w$pubdate),
             pubyear = fix_null(strsplit(w$pubdate, " ")[[1]][1]), 
             journal = fix_null(w$source), 
             authors = fix_null(paste(w$authors$name, collapse = ", ")),
             volume = fix_null(w$volume),
             issue = fix_null(w$issue),
             pages = fix_null(w$pages), 
             doi = fix_null(w$articleids$value[w$articleids$idtype == "doi"]),
             pmid = fix_null(w$articleids$value[w$articleids$idtype == "pubmed"]),
             elocationid = fix_null(w$elocationid),
             stringsAsFactors = FALSE)
})
## Put into data frame and arrange by year
summ <- do.call(rbind, summ) %>% dplyr::arrange(desc(pubyear))
```

```{r}
## Change some HTML formatting to markdown
summ$title <- sapply(summ$title, function(x) {
  x <- gsub("&lt;i&gt;|&lt;/i&gt;", "*", x)  ## <i>, </i>
  x <- gsub("&lt;u&gt;|&lt;/u&gt;", "*", x)  ## <u>, </u>
  x
})
```

```{r}
## Add unlisted publications
unlisted_publications <- unlisted_publications[, match(colnames(summ), 
                                                       colnames(unlisted_publications))]
colnames(unlisted_publications) <- colnames(summ)
summ <- rbind(summ, unlisted_publications)
```


```{r}
## Highlight name in bold
summ$authors <- sapply(summ$authors, function(a) {
  gsub(paste0("(", paste(highlight, collapse = "|"), ")"), "<strong>\\1</strong>", a)
})

summ$authors <- sapply(summ$authors, function(a) {
  gsub(", Bar-Joseph Z", "", a)
})
```

```{r add_links, message = FALSE, warning = FALSE}
## Add column with links to GitHub repos
summ <- dplyr::left_join(summ, data.frame(pmid_github, stringsAsFactors = FALSE)) %>%
  dplyr::mutate(github = replace(github, is.na(github), ""))

## Add column with links to software packages
summ <- dplyr::left_join(summ, data.frame(pmid_software, stringsAsFactors = FALSE)) %>%
    dplyr::mutate(software = replace(software, is.na(software), ""))

## Add column with links to preprints
summ <- dplyr::left_join(summ, data.frame(pmid_preprints, stringsAsFactors = FALSE)) %>%
  dplyr::mutate(preprint = replace(preprint, is.na(preprint), ""))
```

```{r add_citations, message = FALSE, warning = FALSE}
## Add citations from Google Scholar
idx <- match(tolower(gsub(" $", "", gsub("  ", " ", gsub("[[:punct:]]", " ",
                                                         summ$title)))),
             tolower(gsub(" $", "", gsub("  ", " ", gsub("[[:punct:]]", " ",
                                                         scholarpubs$title)))))
summ$citations <- scholarpubs$cites[idx]
```


```{r}
## Add bioRxiv preprints by doi
biorxiv <- do.call(rbind, lapply(biorxiv_dois, function(doi) {
  html <- read_html(paste0("https://doi.org/", doi))
  title <- html_text(html_nodes(html, "#page-title"))
  authors <- paste(unique(paste(html_text(html_nodes(html, ".nlm-given-names")), 
                                html_text(html_nodes(html, ".nlm-surname")))),
                   collapse = ", ")
  #pubdate <- html_text(html_nodes(html, ".published"))
  pubdate <- gsub("^ *| *$", "", gsub("\n", "", html_text(html_nodes(html, ".pane-1 .pane-content"))))
  data.frame(title = title, pubdate = pubdate, pubyear = "Preprints", 
             journal = "bioRxiv", authors = authors, volume = "", issue = "", 
             pages = "", doi = doi, pmid = "", elocationid = "", 
             stringsAsFactors = FALSE)
}))
```

```{r}
## Add PeerJ preprints by doi
peerj <- do.call(rbind, lapply(peerj_dois, function(doi) {
  html <- read_html(paste0("https://doi.org/", doi))
  title <- html_text(html_nodes(html, ".article-title"))
  authors <- paste(unique(paste(html_text(html_nodes(html, ".given-names")), 
                                html_text(html_nodes(html, ".surname")))), 
                   collapse = ", ")
  pubdate <- paste0("Posted ", format(as.Date(html_text(html_nodes(html, "time"))[1]), "%B %d, %Y."))
  data.frame(title = title, pubdate = pubdate, pubyear = "Preprints",       
             journal = "PeerJ", authors = authors, volume = "", issue = "", 
             pages = "", doi = doi, pmid = "", elocationid = "", 
             stringsAsFactors = FALSE)
}))
```

```{r}
## Add ArXiv preprints by doi
arxiv <- do.call(rbind, lapply(arxiv_dois, function(doi) {
  html <- read_html(paste0("https://arxiv.org/abs/", doi))
  title <- gsub("^Title:", "", html_text(html_nodes(html, ".title")))
  authors <- gsub(" *$", "", gsub("\n", "", gsub("^Authors:", "", html_text(html_nodes(html, ".authors")))))
  tmp <- strsplit(html_text(html_nodes(html, ".submission-history")), "\n")[[1]]
  tmp <- strsplit(tmp[length(tmp)], " ")[[1]][c(3, 2, 4)]
  pubdate <- paste0("Posted ", tmp[1], " ", tmp[2], ", ", tmp[3], ".")
  data.frame(title = title, pubdate = pubdate, pubyear = "Preprints",       
             journal = "ArXiv", authors = authors, volume = "", issue = "", 
             pages = "", doi = doi, pmid = "", elocationid = "", 
             stringsAsFactors = FALSE)
}))
```

```{r}
## Add F1000Research preprints
if (!is.null(biorxiv) && !is.null(f1000_preprints)) {
  f1000_preprints <- f1000_preprints[, match(colnames(biorxiv), 
                                             colnames(f1000_preprints))]
  colnames(f1000_preprints) <- colnames(biorxiv)
  biorxiv <- rbind(biorxiv, f1000_preprints)
} else if (!is.null(f1000_preprints)) {
  biorxiv <- f1000_preprints
}

## Add PeerJ preprints
if (!is.null(peerj)) {
  biorxiv <- plyr::rbind.fill(biorxiv, peerj)
}

## Add ArXiv preprints
if (!is.null(arxiv)) {
  biorxiv <- plyr::rbind.fill(biorxiv, arxiv)
}
```

```{r add_links_preprints, message = FALSE, warning = FALSE}
if (!is.null(biorxiv)) {
  ## Add column with links to GitHub repos
  biorxiv <- dplyr::left_join(biorxiv, data.frame(biorxiv_github, stringsAsFactors = FALSE)) %>%
    dplyr::mutate(github = replace(github, is.na(github), ""))
  
  ## Add colum with links to software packages
  biorxiv <- dplyr::left_join(biorxiv, data.frame(biorxiv_software, stringsAsFactors = FALSE)) %>%
    dplyr::mutate(software = replace(software, is.na(software), ""))
}
```

```{r add_citations_preprints, message = FALSE, warning = FALSE}
if (!is.null(biorxiv)) {
  ## Add citations from Google Scholar
  idx <- match(tolower(gsub(" $", "", gsub("  ", " ", gsub("[[:punct:]]", " ",
                                                           biorxiv$title)))),
               tolower(gsub(" $", "", gsub("  ", " ", gsub("[[:punct:]]", " ",
                                                           scholarpubs$title)))))
  biorxiv$citations <- scholarpubs$cites[idx]
}
```


```{r}
if (!is.null(biorxiv)) {
  ## Highlight name in bold
  biorxiv$authors <- sapply(biorxiv$authors, function(a) {
    gsub(paste0("(", paste(highlight, collapse = "|"), ")"), "<strong>\\1</strong>", a)
  })
}
```

```{r}
## Split by publication year
years <- as.character(unique(summ$pubyear))
summ <- split(summ, summ$pubyear)

## Generate final text string to display
txt <- "<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>\n\n"
## Preprints
if (!is.null(biorxiv)) {
  txt <- paste0(txt, "\n## Preprints and articles awaiting peer review\n\n")
  for (j in seq_len(nrow(biorxiv))) {
    if (biorxiv[j, "journal"] != "ArXiv") {
      txt <- paste0(txt, "- ", biorxiv[j, "authors"], ": ", biorxiv[j, "title"], 
                    ". ", biorxiv[j, "journal"], 
                    " doi:", make_link_text(address = paste0("https://doi.org/", biorxiv[j, "doi"]), title = paste0("https://doi.org/", biorxiv[j, "doi"])), ". ", 
                    biorxiv[j, "pubdate"], " ", 
                    ifelse(biorxiv[j, "github"] == "", "", paste0(make_link_text(address = biorxiv[j, "github"], title = "GitHub repo"), ". ")), 
                  ifelse(biorxiv[j, "software"] == "", "", paste0(make_link_text(address = biorxiv[j, "software"], title = "Software"), ". ")),
                  make_link_altmetric(paste0("https://doi.org/", biorxiv[j, "doi"])),
                  "\n\n")
    } else {
      txt <- paste0(txt, "- ", biorxiv[j, "authors"], ": ", biorxiv[j, "title"], 
                  ". ", biorxiv[j, "journal"], 
                  " ID:", make_link_text(address = paste0("https://arxiv.org/abs/", biorxiv[j, "doi"]), title = paste0("https://arxiv.org/abs/", biorxiv[j, "doi"])), ". ", 
                  biorxiv[j, "pubdate"], " ", 
                  ifelse(biorxiv[j, "github"] == "", "", paste0(make_link_text(address = biorxiv[j, "github"], title = "GitHub repo"), ". ")), 
                  ifelse(biorxiv[j, "software"] == "", "", paste0(make_link_text(address = biorxiv[j, "software"], title = "Software"), ". ")),
                  make_link_altmetric_arxiv(biorxiv[j, "doi"]),
                  "\n\n")
    }
  }
}

## Publications
for (i in years) {
  txt <- paste0(txt, "\n## ", i, "\n\n")
  for (j in seq_len(nrow(summ[[i]]))) {
    txt <- paste0(txt, "- ", summ[[i]][j, "authors"], ": ", summ[[i]][j, "title"], 
                  " ", summ[[i]][j, "journal"], " ", summ[[i]][j, "volume"],
                  ifelse(summ[[i]][j, "issue"] == "", "", 
                         paste0("(", summ[[i]][j, "issue"], ")")), ":",
                  summ[[i]][j, "pages"], 
                  " (", i, "). DOI: ", summ[[i]][j, "doi"],
                  ". ", make_link_pmid(pmid = summ[[i]][j, "pmid"]), ". ",  
                  ifelse(summ[[i]][j, "github"] == "", "", paste0(make_link_text(address = summ[[i]][j, "github"], title = "GitHub repo"), ". ")), 
                  ifelse(summ[[i]][j, "software"] == "", "", paste0(make_link_text(address = summ[[i]][j, "software"], title = "Software/Resources"), ". ")), 
                  ifelse(summ[[i]][j, "preprint"] == "", "", paste0(make_link_text(address = summ[[i]][j, "preprint"], title = "Preprint"), ". ")), 
                  make_link_altmetric(paste0("https://doi.org/", summ[[i]][j, "doi"])),
                  "\n\n")
  }
}
```

```{r, results = "asis"}
cat(txt)
```

