<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="" />


<title>Resources</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Charlotte Soneson</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-pdf-o"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="resources.html">
    <span class="fa fa-cogs"></span>
     
    Resources
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-mortar-board"></span>
     
    Teaching
  </a>
</li>
<li>
  <a href="https://github.com/csoneson">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
<li>
  <a href="https://twitter.com/CSoneson">
    <span class="fa fa-twitter"></span>
     
    Twitter
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Resources</h1>

</div>


<center>
<a href="http://bioconductor.org/packages/alevinQC/"><img src="stickers/alevinQC.png" style="width:3cm" /></a><a href="https://github.com/csoneson/ARMOR"><img src="stickers/ARMOR.png" style="width:3cm" /></a><a href="https://bioconductor.org/packages/countsimQC/"><img src="stickers/countsimQC.png" style="width:3cm" /></a><a href="http://bioconductor.org/packages/eisaR/"><img src="stickers/eisaR.png" style="width:3cm" /></a><a href="http://bioconductor.org/packages/ExploreModelMatrix/"><img src="stickers/emm_logo.png" style="width:3cm" /></a><a href="http://bioconductor.org/packages/iCOBRA/"><img src="stickers/iCOBRA.png" style="width:3cm" /></a><a href="http://bioconductor.org/packages/iSEE/"><img src="stickers/iSEE.png" style="width:3cm" /></a>
</center>
<div id="r-packages" class="section level2">
<h2>R packages</h2>
<p>I am a maintainer or contributor of the following R packages:</p>
<ul>
<li><a href="http://bioconductor.org/packages/alevinQC/"><img src="https://img.shields.io/badge/-alevinQC-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/alevinQC.svg">
<ul>
<li>A package for generation of QC reports summarizing the output from an <a href="https://salmon.readthedocs.io/en/latest/alevin.html">alevin</a> run. Reports can be generated as html or pdf files, or as shiny applications.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/compcodeR/"><img src="https://img.shields.io/badge/-compcodeR-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/compcodeR.svg">
<ul>
<li>A package providing extensive functionality for comparing results obtained by different methods for differential expression analysis of RNAseq data. It also contains functions for simulating count data and interfaces to several packages for performing the differential expression analysis.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/countsimQC/"><img src="https://img.shields.io/badge/-countsimQC-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/countsimQC.svg">
<ul>
<li>countsimQC provides functionality to create a comprehensive report comparing a broad range of characteristics across a collection of count matrices. One important use case is the comparison of one or more synthetic count matrices to a real count matrix, possibly the one underlying the simulations. However, any collection of count matrices can be compared.</li>
</ul></li>
<li><a href="https://github.com/csoneson/dreval"><img src="https://img.shields.io/badge/-dreval-red.svg" /></a>
<ul>
<li>Convenience functions for evaluation and comparison of reduced dimension representations.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/eisaR/"><img src="https://img.shields.io/badge/-eisaR-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/eisaR">
<ul>
<li>Exon-intron split analysis (EISA) uses ordinary RNA-seq data to measure changes in mature RNA and pre-mRNA reads across different experimental conditions to quantify transcriptional and post-transcriptional regulation of gene expression.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/ExploreModelMatrix/"><img src="https://img.shields.io/badge/-ExploreModelMatrix-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/ExploreModelMatrix.svg">
<ul>
<li>Interactive interface for exploring design matrices in R.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/iCOBRA/"><img src="https://img.shields.io/badge/-iCOBRA-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/iCOBRA.svg">
<ul>
<li>iCOBRA provides functions for calculation and visualization of performance metrics for evaluation of ranking and binary classification (assignment) methods. It also contains a shiny application for interactive exploration of results.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/iSEE/"><img src="https://img.shields.io/badge/-iSEE-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/iSEE.svg">
<ul>
<li>iSEE provides functions for creating an interactive Shiny-based graphical user interface for exploring data stored in SummarizedExperiment objects, including row- and column-level metadata. Particular attention is given to single-cell data in a SingleCellExperiment object with visualization of dimensionality reduction results.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/iSEEu/"><img src="https://img.shields.io/badge/-iSEEu-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/iSEEu.svg">
<ul>
<li>iSEEu (the iSEE universe) contains diverse functionality to extend the usage of the iSEE package, including additional classes for the panels, or modes allowing easy configuration of iSEE applications.</li>
</ul></li>
<li><a href="https://github.com/csoneson/jcc"><img src="https://img.shields.io/badge/-jcc-red.svg" /></a>
<ul>
<li>Calculates <a href="http://www.life-science-alliance.org/content/2/1/e201800175.abstract">JCC scores</a>, which are aimed at detecting genes for which estimated transcript abundances are unreliable, either because of problems in the transcript abundance estimation or because of missing or wrongly annotated reference transcripts. It does so by comparing the number of reads predicted to align across each junction, inferred from the transcript abundances and a fragment bias model, to the observed number of junction-spanning reads, obtained via alignment of the reads to the genome. A high JCC score for a gene indicates that the estimated abundances for the corresponding transcripts are unreliable and should be treated with care in downstream analyses.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/Rhisat2/"><img src="https://img.shields.io/badge/-Rhisat2-blue.svg" /></a> <img src="https://bioconductor.org/shields/years-in-bioc/Rhisat2.svg">
<ul>
<li>An R interface to the HISAT2 spliced short-read aligner by Kim et al. (2015). The package contains wrapper functions to create a genome index and to perform the read alignment to the generated index.</li>
</ul></li>
</ul>
</div>
<div id="other-resources" class="section level2">
<h2>Other resources</h2>
<ul>
<li><a href="https://github.com/csoneson/ARMOR"><img src="https://img.shields.io/badge/-ARMOR-red.svg" /></a>
<ul>
<li>A lightweight Snakemake workflow for preprocessing and statistical analysis of RNA-seq data.</li>
</ul></li>
<li><a href="http://imlspenticton.uzh.ch:3838/conquer/"><img src="https://img.shields.io/badge/-conquer-red.svg" /></a>
<ul>
<li>A repository with consistently quantified single-cell RNA-seq data sets.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/TabulaMurisData/"><img src="https://img.shields.io/badge/-TabulaMurisData-blue.svg" /></a>
<ul>
<li>An R package interface to processed 10x (droplet) and SmartSeq2 (on FACS-sorted cells) single-cell RNA-seq data from the <a href="http://tabula-muris.ds.czbiohub.org/">Tabula Muris consortium</a>.</li>
</ul></li>
<li><a href="http://bioconductor.org/packages/DuoClustering2018/"><img src="https://img.shields.io/badge/-DuoClustering2018-blue.svg" /></a>
<ul>
<li>Data sets and clustering results from the scRNA-seq clustering evaluation by <a href="https://f1000research.com/articles/7-1141/v2">Duo et al. 2018</a>.</li>
</ul></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
