<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="" />


<title>Teaching</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Charlotte Soneson</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-pdf-o"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="resources.html">
    <span class="fa fa-cogs"></span>
     
    Resources
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-mortar-board"></span>
     
    Teaching
  </a>
</li>
<li>
  <a href="https://github.com/csoneson">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
<li>
  <a href="https://twitter.com/CSoneson">
    <span class="fa fa-twitter"></span>
     
    Twitter
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Teaching</h1>

</div>


<div id="section" class="section level2">
<h2>2020</h2>
<ul>
<li><a href="https://www.huber.embl.de/csama2020/#home"><del>Statistical data analysis for genome scale biology (CSAMA 2020)</del></a>, Brixen, Italy (cancelled)</li>
<li><a href="https://uclouvain-cbio.github.io/BSS2020/"><del>Bioinformatics summer school</del></a>, Louvain-la-Neuve, Belgium (cancelled)</li>
<li><a href="https://www.sib.swiss/training/course/2020-05-adv-scrna">Advanced topics in single-cell transcriptomics</a>, Basel, Switzerland (streamed)</li>
<li><del>RNA-seq analysis</del>, University of Freiburg, Germany (postponed)</li>
<li><a href="https://ivanek.github.io/analysisOfGenomicsDataWithR/">Analysis of genomics data with R/Bioconductor</a>, University of Basel, Switzerland</li>
</ul>
</div>
<div id="section-1" class="section level2">
<h2>2019</h2>
<ul>
<li><a href="https://nbisweden.github.io/single-cell_sib_scilifelab/README.html">Single-cell analysis autumn school</a>, Leysin, Switzerland</li>
<li><a href="https://ivanek.github.io/introductionToR/">Introduction to R</a>, University of Basel, Switzerland</li>
<li><a href="https://www.huber.embl.de/csama2019/#home">Statistical data analysis for genome scale biology (CSAMA 2019)</a>, Brixen, Italy</li>
<li><a href="https://uclouvain-cbio.github.io/BSS2019/">Bioinformatics summer school</a>, Louvain-la-Neuve, Belgium</li>
<li><a href="https://www.ebi.ac.uk/training/events/2019/rna-sequence-analysis">RNA-sequence analysis</a>, EBI, Hinxton, UK</li>
<li><a href="https://ivanek.github.io/analysisOfGenomicsDataWithR/">Analysis of genomics data with R/Bioconductor</a>, University of Basel, Switzerland</li>
<li><a href="http://gdc-mda.ethz.ch/MDA19/program.html">Microbiota Data Analysis Workshop</a>, Zurich, Switzerland</li>
</ul>
</div>
<div id="section-2" class="section level2">
<h2>2018</h2>
<ul>
<li><a href="http://www-huber.embl.de/csama2018/#home">Statistical data analysis for genome scale biology (CSAMA 2018)</a>, Brixen, Italy</li>
<li><a href="https://www.ebi.ac.uk/training/events/2018/advanced-rna-seq-analysis">Advanced RNA-seq analysis</a>, EBI, Hinxton, UK</li>
</ul>
</div>
<div id="section-3" class="section level2">
<h2>2017</h2>
<ul>
<li><a href="https://www.sib.swiss/training/course/stats-basel2017">Statistics for Life Scientists</a>, Basel, Switzerland</li>
<li><a href="http://www.huber.embl.de/csama2017/#home">Statistical data analysis for genome scale biology (CSAMA 2017)</a>, Brixen, Italy</li>
<li><a href="http://www.ebi.ac.uk/training/events/2017/advanced-rna-seq-and-chip-seq-data-analysis-1">Advanced RNA-seq and ChIP-seq data analysis</a>, EBI, Hinxton, UK</li>
</ul>
</div>
<div id="section-4" class="section level2">
<h2>2016</h2>
<ul>
<li>Advanced course on edgeR and DESeq2, Leuven, Belgium</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=269">Statistics for Life Scientists</a>, Basel, Switzerland</li>
<li><a href="http://www.uni-regensburg.de/medicine/statistical-bioinformatics/method-course-oct-16/program/index.html">Method course - Analysis of NextGen RNA-seq data for expression profiling and protein-binding RNAs</a>, Regensburg, Germany</li>
<li><a href="http://events.embo.org/16-htp-seq/">Analysis of high-throughput sequencing data</a>, Cambridge, UK</li>
<li><a href="http://www.huber.embl.de/csama2016/about/">Statistics and computing in genome data science (CSAMA 2016)</a>, Brixen, Italy</li>
</ul>
</div>
<div id="section-5" class="section level2">
<h2>2015</h2>
<ul>
<li>Statistical transcriptomics data analysis: from design to reproducible research, Fribourg, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=189">Statistical techniques for genomics/life sciences</a>, Basel, Switzerland</li>
</ul>
</div>
<div id="section-6" class="section level2">
<h2>2014</h2>
<ul>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=187">Multivariate analysis for exploration and visualisation of biological data</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=186">Introduction to classification</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=177">Advanced statistics:statistical modeling</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=175">Microarray and RNAseq data analysis with R</a>, Fribourg, Switzerland</li>
<li>RNA-seq data analysis, Brno, Czech Republic</li>
<li><a href="http://www.bc2.ch/courses/RNASeqCourseBasel201401/">RNA-seq: from experiment planning to statistical data analysis</a>, Basel, Switzerland</li>
</ul>
</div>
<div id="section-7" class="section level2">
<h2>2013</h2>
<ul>
<li><a href="http://biologie.cuso.ch/staromics/activities/detail-activity/item/courses/rna-seq-from-experiment-planing-to-statistical-data-analysis/">RNA-seq: from experiment planning to statistical data analysis</a>, Bern, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=150">Multivariate analysis for exploration and visualisation of biological data</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=146">Advanced transcriptomics analysis</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=147">Introduction to statistics for biologists</a>, Lausanne, Switzerland</li>
</ul>
</div>
<div id="section-8" class="section level2">
<h2>2012</h2>
<ul>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=124">Integration of data from different omics</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=120">Analysis of differential gene expression</a>, Lausanne, Switzerland</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=121">Multivariate analysis for exploration and visualisation of biological data</a>, Lausanne, Switzerland</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
