<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.45">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Tools – Charlotte Soneson</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-listing/list.min.js"></script>
<script src="../site_libs/quarto-listing/quarto-listing.js"></script>
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script>

  window.document.addEventListener("DOMContentLoaded", function (_event) {
    const listingTargetEl = window.document.querySelector('#listing-listing .list');
    if (!listingTargetEl) {
      // No listing discovered, do not attach.
      return; 
    }

    const options = {
      valueNames: ['listing-package','listing-image','listing-description','listing-role',{ data: ['index'] },{ data: ['categories'] },{ data: ['listing-package-sort'] }],
      
      searchColumns: ["listing-package","listing-package_lower","listing-image","listing-role","listing-description"],
    };

    window['quarto-listings'] = window['quarto-listings'] || {};
    window['quarto-listings']['listing-listing'] = new List('listing-listing', options);

    if (window['quarto-listing-loaded']) {
      window['quarto-listing-loaded']();
    }
  });

  window.addEventListener('hashchange',() => {
    if (window['quarto-listing-loaded']) {
      window['quarto-listing-loaded']();
    }
  })
  </script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="../styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Charlotte Soneson</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../index.html"> <i class="bi bi-fa-home" role="img">
</i> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../publications.html"> <i class="bi bi-fa-file-pdf-o" role="img">
</i> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../tools/index.html" aria-current="page"> <i class="bi bi-fa-cogs" role="img">
</i> 
<span class="menu-text">Tools</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../teaching.html"> <i class="bi bi-fa-mortar-board" role="img">
</i> 
<span class="menu-text">Teaching</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Tools</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>This page lists tools for which I am involved as a maintainer, co-developer or occasional contributor.</p>




<div class="quarto-listing quarto-listing-container-table" id="listing-listing">
<table class="quarto-listing-table table">
<thead>
<tr>
<th>
Package
</th>
<th>
</th>
<th>
Description
</th>
<th>
Role
</th>
</tr>
</thead>
<tbody class="list">
<tr data-index="0" data-listing-package-sort="alevinQC">
<td>
<a href="https://bioconductor.org/packages/alevinQC/" class="package listing-package">alevinQC</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/alevinQC.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">A package for generation of QC reports summarizing the output from an <a href="https://salmon.readthedocs.io/en/latest/alevin.html">alevin</a> or <a href="https://alevin-fry.readthedocs.io/en/latest/">alevin-fry</a> run. Reports can be generated as html or pdf files, or as Shiny applications. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="1" data-listing-package-sort="ARMOR">
<td>
<a href="https://github.com/csoneson/ARMOR" class="package listing-package">ARMOR</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/ARMOR.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">A light-weight Snakemake workflow for preprocessing and statistical analysis of RNA-seq data. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="2" data-listing-package-sort="bettr">
<td>
<a href="https://bioconductor.org/packages/bettr/" class="package listing-package">bettr</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/bettr.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">bettr provides a set of interactive visualization methods to explore the results of a benchmarking study, where typically more than a single performance measure is computed. The user can weight the performance measures according to their preferences. Performance measures can also be grouped and aggregated according to additional annotations. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="3" data-listing-package-sort="BiocPkgTools">
<td>
<a href="http://bioconductor.org/packages/BiocPkgTools/" class="package listing-package">BiocPkgTools</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/BiocPkgTools.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Bioconductor has a rich ecosystem of metadata around packages, usage, and build status. This package is a simple collection of functions to access that metadata from R. The goal is to expose metadata for data mining and value-added functionality such as package searching, text mining, and analytics on packages. </span>
</td>
<td>
<span class="listing-role">contributor</span>
</td>
</tr>
<tr data-index="4" data-listing-package-sort="compcodeR">
<td>
<a href="https://bioconductor.org/packages/compcodeR/" class="package listing-package">compcodeR</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">A package providing functionality for comparing results obtained by different methods for differential expression analysis of RNAseq data. It also contains functions for simulating count data and interfaces to several packages for performing the differential expression analysis. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="5" data-listing-package-sort="ConfoundingExplorer">
<td>
<a href="https://github.com/csoneson/ConfoundingExplorer" class="package listing-package">ConfoundingExplorer</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">The ConfoundingExplorer R package provides a simple Shiny app for interactively exploring the effect of confounding between a group variable of interest and a batch variable, in terms of the ability to correctly find the variables that are truly differential between the different levels of the group variable. It is mainly intended for teaching purposes and to illustrate important concepts in experimental design, and all analyses are performed using (quite simplistic) simulated data. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="6" data-listing-package-sort="countsimQC">
<td>
<a href="https://bioconductor.org/packages/countsimQC/" class="package listing-package">countsimQC</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/countsimQC.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">countsimQC provides functionality to create a comprehensive report comparing a broad range of characteristics across a collection of count matrices. One important use case is the comparison of one or more synthetic count matrices to a real count matrix, possibly the one underlying the simulations. However, any collection of count matrices can be compared. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="7" data-listing-package-sort="dreval">
<td>
<a href="https://github.com/csoneson/dreval" class="package listing-package">dreval</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">Convenience functions for evaluation and comparison of reduced dimension representations. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="8" data-listing-package-sort="DuoClustering2018">
<td>
<a href="http://bioconductor.org/packages/DuoClustering2018/" class="package listing-package">DuoClustering2018</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">Data sets and clustering results from the scRNA-seq clustering evaluation by <a href="https://f1000research.com/articles/7-1141/v3">Duo et al.&nbsp;2018</a>. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="9" data-listing-package-sort="einprot">
<td>
<a href="https://github.com/fmicompbio/einprot" class="package listing-package">einprot</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/einprot.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">einprot contains a set of workflows for statistical analysis of quantitative proteomics data processed with MaxQuant (LFQ), FragPipe (LFQ), Proteome Discoverer (TMT-labelled), Spectronaut or DIA-NN. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="10" data-listing-package-sort="eisaR">
<td>
<a href="https://bioconductor.org/packages/eisaR/" class="package listing-package">eisaR</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/eisaR.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Exon-intron split analysis (EISA) uses ordinary RNA-seq data to measure changes in mature RNA and pre-mRNA reads across different experimental conditions to quantify transcriptional and post-transcriptional regulation of gene expression. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="11" data-listing-package-sort="ExploreModelMatrix">
<td>
<a href="https://bioconductor.org/packages/ExploreModelMatrix/" class="package listing-package">ExploreModelMatrix</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/emm_logo.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Given a sample data table and a design formula, ExploreModelMatrix generates an interactive application for exploration of the resulting design matrix. This can be helpful for interpreting model coefficients and constructing appropriate contrasts in (generalized) linear models. Static visualizations can also be generated. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="12" data-listing-package-sort="ez-zarr">
<td>
<a href="https://pypi.org/project/ez-zarr/" class="package listing-package">ez-zarr</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/ez_zarr.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">The aim of ez-zarr is to provide easy, high-level access and visualization of OME-Zarr filesets stored according to the NGFF specifications. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="13" data-listing-package-sort="ezzarr">
<td>
<a href="https://github.com/fmicompbio/ezzarr" class="package listing-package">ezzarr</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">The ezzarr R package provides a thin wrapper around the ez-zarr python package for easier use from R. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="14" data-listing-package-sort="footprintR">
<td>
<a href="https://github.com/fmicompbio/footprintR" class="package listing-package">footprintR</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/footprintR_logo.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">footprintR provides tools for working with single-molecule footprinting data in R. These include functions for reading base modifications from bam files or files generated by modkit, efficient representation of such data as R objects, and functions to manipulate and visualize such objects. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="15" data-listing-package-sort="HDCytoData">
<td>
<a href="http://bioconductor.org/packages/HDCytoData/" class="package listing-package">HDCytoData</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">Data package containing a set of publicly available high-dimensional cytometry benchmark datasets, formatted into SummarizedExperiment and flowSet Bioconductor object formats, including all required metadata. Row metadata includes sample IDs, group IDs, patient IDs, reference cell population or cluster labels (where available), and labels identifying ‘spiked in’ cells (where available). Column metadata includes channel names, protein marker names, and protein marker classes (cell type or cell state). </span>
</td>
<td>
<span class="listing-role">contributor</span>
</td>
</tr>
<tr data-index="16" data-listing-package-sort="iCOBRA">
<td>
<a href="https://bioconductor.org/packages/iCOBRA/" class="package listing-package">iCOBRA</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/iCOBRA.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">iCOBRA provides functions for calculation and visualization of performance metrics for evaluation of ranking and binary classification (assignment) methods. It also contains a Shiny application for interactive exploration of results. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="17" data-listing-package-sort="iSEE">
<td>
<a href="https://bioconductor.org/packages/iSEE/" class="package listing-package">iSEE</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/iSEE.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">iSEE allows the user to create an interactive Shiny-based graphical user interface for exploring data stored in SummarizedExperiment objects, including row- and column-level metadata. The interface supports transmission of selections between plots and tables, code tracking, interactive tours, interactive or programmatic initialization, preservation of app state, and extensibility to new panel types via S4 classes. Special attention is given to single-cell data in a SingleCellExperiment object with visualization of dimensionality reduction results. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="18" data-listing-package-sort="iSEEpathways">
<td>
<a href="http://bioconductor.org/packages/iSEEpathways/" class="package listing-package">iSEEpathways</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">This package contains diverse functionality to extend the usage of the iSEE package, including additional classes for the panels or modes facilitating the analysis of pathway analysis results. The package does not perform pathway analysis. Instead, it provides methods to embed precomputed pathway analysis results in a SummarizedExperiment object, in a manner that is compatible with interactive visualisation in iSEE applications. </span>
</td>
<td>
<span class="listing-role">contributor</span>
</td>
</tr>
<tr data-index="19" data-listing-package-sort="iSEEu">
<td>
<a href="https://bioconductor.org/packages/iSEEu/" class="package listing-package">iSEEu</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/iSEEu.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">iSEEu (the iSEE universe) contains diverse functionality to extend the usage of the iSEE package, including additional panel classes and modes allowing easy configuration of iSEE applications. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="20" data-listing-package-sort="jcc">
<td>
<a href="https://github.com/csoneson/jcc" class="package listing-package">jcc</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">This package calculates JCC scores, which are aimed at detecting genes for which estimated transcript abundances are unreliable, either because of problems in the transcript abundance estimation or because of missing or wrongly annotated reference transcripts. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="21" data-listing-package-sort="monaLisa">
<td>
<a href="http://bioconductor.org/packages/monaLisa/" class="package listing-package">monaLisa</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/monaLisa.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Useful functions to work with sequence motifs in the analysis of genomics data. These include methods to annotate genomic regions or sequences with predicted motif hits and to identify motifs that drive observed changes in accessibility or expression. Functions to produce informative visualizations of the obtained results are also provided. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="22" data-listing-package-sort="muscat">
<td>
<a href="http://bioconductor.org/packages/muscat/" class="package listing-package">muscat</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/muscat.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">muscat provides various methods and visualization tools for differential state analysis in multi-sample, multi-group, multi-(cell-)subpopulation scRNA-seq data, including cell-level mixed models and methods based on aggregated “pseudobulk” data, as well as a flexible simulation platform that mimics both single- and multi-sample scRNA-seq data. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="23" data-listing-package-sort="mutscan">
<td>
<a href="https://github.com/fmicompbio/mutscan" class="package listing-package">mutscan</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/mutscan.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">mutscan provides functionality for processing and statistical analysis of multiplexed assays of variant effect (MAVE) and similar data. The package contains functions covering the full workflow from raw FASTQ files to publication-ready visualizations. A broad range of library designs can be processed with a single, unified interface. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="24" data-listing-package-sort="orthos">
<td>
<a href="http://bioconductor.org/packages/orthos/" class="package listing-package">orthos</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/orthos.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">orthos decomposes RNA-seq contrasts, for example obtained from a gene knock-out or compound treatment experiment, into unspecific and experiment-specific components. Original and decomposed contrasts can be efficiently queried against a large database of contrasts (derived from ARCHS4, https://maayanlab.cloud/archs4/) to identify similar experiments. orthos furthermore provides plotting functions to visualize the results of such a search for similar contrasts. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="25" data-listing-package-sort="QuasR">
<td>
<a href="http://bioconductor.org/packages/QuasR/" class="package listing-package">QuasR</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/quasr.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">QuasR provides a framework for the quantification and analysis of short reads. It covers a complete workflow starting from raw sequence reads, over creation of alignments and quality control plots, to the quantification of genomic regions of interest. Read alignments are either generated through Rbowtie (data from DNA/ChIP/ATAC/Bis-seq experiments) or Rhisat2 (data from RNA-seq experiments that require spliced alignments), or can be provided in the form of bam files. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="26" data-listing-package-sort="Rhisat2">
<td>
<a href="http://bioconductor.org/packages/Rhisat2/" class="package listing-package">Rhisat2</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">An R interface to the HISAT2 spliced short-read aligner by Kim et al.&nbsp;(2015). The package contains wrapper functions to create a genome index and to perform the read alignment to the generated index. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="27" data-listing-package-sort="sketchR">
<td>
<a href="https://bioconductor.org/packages/sketchR/" class="package listing-package">sketchR</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/sketchR.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Provides an R interface for various subsampling algorithms implemented in python packages. Currently, interfaces to the geosketch and scSampler python packages are implemented. In addition it also provides diagnostic plots to evaluate the subsampling. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="28" data-listing-package-sort="swissknife">
<td>
<a href="https://github.com/fmicompbio/swissknife" class="package listing-package">swissknife</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">Handy code shared in the FMI CompBio group. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="29" data-listing-package-sort="TabulaMurisData">
<td>
<a href="http://bioconductor.org/packages/TabulaMurisData/" class="package listing-package">TabulaMurisData</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">An R package interface to processed 10x (droplet) and SmartSeq2 (on FACS-sorted cells) single-cell RNA-seq data from the Tabula Muris consortium. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="30" data-listing-package-sort="TabulaMurisSenisData">
<td>
<a href="http://bioconductor.org/packages/TabulaMurisSenisData/" class="package listing-package">TabulaMurisSenisData</a>
</td>
<td>
<span class="listing-image">&nbsp;</span>
</td>
<td>
<span class="listing-description">An R package interface to processed 10x (droplet) and SmartSeq2 (on FACS-sorted cells) single-cell RNA-seq data from the Tabula Muris Senis project. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="31" data-listing-package-sort="tidySummarizedExperiment">
<td>
<a href="https://www.bioconductor.org/packages/tidySummarizedExperiment/" class="package listing-package">tidySummarizedExperiment</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/tidyomics.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">The tidySummarizedExperiment package provides a set of tools for creating and manipulating tidy data representations of SummarizedExperiment objects. </span>
</td>
<td>
<span class="listing-role">contributor</span>
</td>
</tr>
<tr data-index="32" data-listing-package-sort="treeclimbR">
<td>
<a href="https://bioconductor.org/packages/treeclimbR/" class="package listing-package">treeclimbR</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/treeclimbR.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">The arrangement of hypotheses in a hierarchical structure appears in many research fields and often indicates different resolutions at which data can be viewed. This raises the question of which resolution level the signal should best be interpreted on. treeclimbR provides a flexible method to select optimal resolution levels (potentially different levels in different parts of the tree), rather than cutting the tree at an arbitrary level. </span>
</td>
<td>
<span class="listing-role">maintainer</span>
</td>
</tr>
<tr data-index="33" data-listing-package-sort="tximeta">
<td>
<a href="http://bioconductor.org/packages/tximeta/" class="package listing-package">tximeta</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/tximeta.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Transcript quantification import from Salmon and other quantifiers with automatic attachment of transcript ranges and release information, and other associated metadata. De novo transcriptomes can be linked to the appropriate sources with linkedTxomes and shared for computational reproducibility. </span>
</td>
<td>
<span class="listing-role">contributor</span>
</td>
</tr>
<tr data-index="34" data-listing-package-sort="tximport">
<td>
<a href="http://bioconductor.org/packages/tximport/" class="package listing-package">tximport</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/tximport.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">Imports transcript-level abundances, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages. Average transcript length, weighted by sample-specific transcript abundance estimates, is provided as a matrix which can be used as an offset for differential expression of gene-level counts. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
<tr data-index="35" data-listing-package-sort="velociraptor">
<td>
<a href="https://bioconductor.org/packages/velociraptor/" class="package listing-package">velociraptor</a>
</td>
<td>
<span class="listing-image"><img loading="lazy" src="stickers/velociraptor_sticker.png" style="height: 80px;"></span>
</td>
<td>
<span class="listing-description">This package provides Bioconductor-friendly wrappers for RNA velocity calculations in single-cell RNA-seq data. We use the basilisk package to manage Conda environments, and the zellkonverter package to convert data structures between SingleCellExperiment (R) and AnnData (Python). The information produced by the velocity methods is stored in the various components of the SingleCellExperiment class. </span>
</td>
<td>
<span class="listing-role">co-developer</span>
</td>
</tr>
</tbody>
</table>
<div class="listing-no-matching d-none">
No matching items
</div>
</div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>