<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.45">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Publications – Charlotte Soneson</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Charlotte Soneson</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> <i class="bi bi-fa-home" role="img">
</i> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./publications.html" aria-current="page"> <i class="bi bi-fa-file-pdf-o" role="img">
</i> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./resources.html"> <i class="bi bi-fa-cogs" role="img">
</i> 
<span class="menu-text">Resources</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./teaching.html"> <i class="bi bi-fa-mortar-board" role="img">
</i> 
<span class="menu-text">Teaching</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/csoneson"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://fosstodon.org/@csoneson"> <i class="bi bi-mastodon" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#preprints-and-articles-awaiting-peer-review" id="toc-preprints-and-articles-awaiting-peer-review" class="nav-link active" data-scroll-target="#preprints-and-articles-awaiting-peer-review">Preprints and articles awaiting peer review</a></li>
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section">2024</a></li>
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1">2023</a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2">2022</a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3">2021</a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4">2020</a></li>
  <li><a href="#section-5" id="toc-section-5" class="nav-link" data-scroll-target="#section-5">2019</a></li>
  <li><a href="#section-6" id="toc-section-6" class="nav-link" data-scroll-target="#section-6">2018</a></li>
  <li><a href="#section-7" id="toc-section-7" class="nav-link" data-scroll-target="#section-7">2017</a></li>
  <li><a href="#section-8" id="toc-section-8" class="nav-link" data-scroll-target="#section-8">2016</a></li>
  <li><a href="#section-9" id="toc-section-9" class="nav-link" data-scroll-target="#section-9">2015</a></li>
  <li><a href="#section-10" id="toc-section-10" class="nav-link" data-scroll-target="#section-10">2014</a></li>
  <li><a href="#section-11" id="toc-section-11" class="nav-link" data-scroll-target="#section-11">2013</a></li>
  <li><a href="#section-12" id="toc-section-12" class="nav-link" data-scroll-target="#section-12">2012</a></li>
  <li><a href="#section-13" id="toc-section-13" class="nav-link" data-scroll-target="#section-13">2011</a></li>
  <li><a href="#section-14" id="toc-section-14" class="nav-link" data-scroll-target="#section-14">2010</a></li>
  <li><a href="#section-15" id="toc-section-15" class="nav-link" data-scroll-target="#section-15">2009</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Publications</h1>
<p class="subtitle lead"><a href="https://scholar.google.ch/citations?user=c28GqkIAAAAJ&amp;hl=en">Google Scholar page</a></p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<!--
PubMed-indexed preprints are indicated with a title in <span style="color:MediumOrchid;">purple</span>. 
-->
<script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
<section id="preprints-and-articles-awaiting-peer-review" class="level2">
<h2 class="anchored" data-anchor-id="preprints-and-articles-awaiting-peer-review">Preprints and articles awaiting peer review</h2>
<ul>
<li><p>Skribbe, M.; <strong>Soneson, C.</strong>; Stadler, M. B.; Schwaiger, M.; Suma Sreechakram, V. N.; Iesmantavicius, V.; Hess, D.; Pandini Figueiredo Moreno, E.; Braun, S.; Seebacher, J.; Smallwood, S. A.; Buhler, M.: A comprehensive Schizosaccharomyces pombe atlas of physical transcription factor interactions with proteins and chromatin. bioRxiv doi:<a href="https://doi.org/10.1101/2024.08.20.607873" target="_blank"> https://doi.org/10.1101/2024.08.20.607873</a>. Posted August 21, 2024. <a href="https://github.com/fmicompbio/Spombe_TFome" target="_blank"> GitHub repo</a>.</p></li>
<li><p>Shi, S.; Iesmantavicius, V.; Sabu, A. S.; <strong>Soneson, C.</strong>; Kohler, H.; Sanchez, J.; Ghosh, S.; Cao, C.; Huang, Y.; Matthias, G.; Yamauchi, Y.; Matthias, P.; Wang, L.: Ubiquitination mediates protein localization in RNA virus-infected cells. bioRxiv doi:<a href="https://doi.org/10.1101/2024.08.15.608034" target="_blank"> https://doi.org/10.1101/2024.08.15.608034</a>. Posted August 15, 2024.</p></li>
<li><p>Gudipati, R. K.; Gaidatzis, D.; Seebacher, J.; Muehlhaeusser, S.; Kempf, G.; Cavadini, S.; Hess, D.; <strong>Soneson, C.</strong>; Grosshans, H.: Massively parallel quantification of substrate turnover defines protease subsite cooperativity. bioRxiv doi:<a href="https://doi.org/10.1101/2023.11.17.567387" target="_blank"> https://doi.org/10.1101/2023.11.17.567387</a>. Posted August 07, 2024.</p></li>
<li><p>Kocere, A.; Chiavacci, E.; <strong>Soneson, C.</strong>; Wells, H. H.; Mendez-Acevedo, K. M.; Macgowan, J. S.; Jacobson, S. T.; Hiltabidle, M. S.; Raghunath, A.; Shavit, J. A.; Panakova, D.; Williams, M. L. K.; Robinson, M. D.; Mosimann, C.; Burger, A.: Rbm8a deficiency causes hematopoietic defects by modulating Wnt/PCP signaling. bioRxiv doi:<a href="https://doi.org/10.1101/2023.04.12.536513" target="_blank"> https://doi.org/10.1101/2023.04.12.536513</a>. Posted February 01, 2024.</p></li>
<li><p>He, D.; <strong>Soneson, C.</strong>; Patro, R.: Understanding and evaluating ambiguity in single-cell and single-nucleus RNA-sequencing. bioRxiv doi:<a href="https://doi.org/10.1101/2023.01.04.522742" target="_blank"> https://doi.org/10.1101/2023.01.04.522742</a>. Posted January 04, 2023. <a href="https://github.com/COMBINE-lab/scrna-ambiguity" target="_blank"> GitHub repo</a>.</p></li>
<li><p>Srivastava, A.; Zakeri, M.; Sarkar, H.; <strong>Soneson, C.</strong>; Kingsford, C.; Patro, R.: Accounting for fragments of unexpected origin improves transcript quantification in RNA-seq simulations focused on increased realism. bioRxiv doi:<a href="https://doi.org/10.1101/2021.01.17.426996" target="_blank"> https://doi.org/10.1101/2021.01.17.426996</a>. Posted January 19, 2021.</p></li>
<li><p>Paulo Czarnewski, Ahmed Mahfouz, Raffaele A. Calogero, Patricia M. Palagi, Laura Portell-Silva, Asier Gonzalez-Uriarte, <strong>Charlotte Soneson</strong>, Tony Burdett, Barbara Szomolay, Pavankumar Videm, Hans-Rudolf Hotz, Irene Papatheodorou, John M. Hancock, Björn Grüning, Wilfried Haerty, Roland Krause, Salvador Capella-Gutierrez, Brane Leskošek, Luca Alessandri, Maddalena Arigoni, Tadeja Rezen, Alexander Botzki, Polonca Ferk, Jessica Lindvall, Katharina F Heil, Naveed Ishaque, Eija Korpelainen: Community-driven ELIXIR activities in single-cell omics. F1000Research doi:<a href="https://doi.org/10.12688/f1000research.122312.1" target="_blank"> https://doi.org/10.12688/f1000research.122312.1</a>. Posted July 29, 2022.</p></li>
<li><p>Izaskun Mallona, Almut Luetge, <strong>Charlotte Soneson</strong>, Ben Carrillo, Reto Gerber, Daniel Incicau, Anthony Sonrel, Mark D. Robinson: Building a continuous benchmarking ecosystem in bioinformatics. ArXiv ID:<a href="https://arxiv.org/abs/2409.15472" target="_blank"> https://arxiv.org/abs/2409.15472</a>. Posted Sep 23, 2024.</p></li>
<li><p>Izaskun Mallona, Almut Luetge, Ben Carrillo, Daniel Incicau, Reto Gerber, Anthony Sonrel, <strong>Charlotte Soneson</strong>, Mark D. Robinson: Omnibenchmark (alpha) for continuous and open benchmarking in bioinformatics. ArXiv ID:<a href="https://arxiv.org/abs/2409.17038" target="_blank"> https://arxiv.org/abs/2409.17038</a>. Posted Sep 25, 2024.</p></li>
</ul>
</section>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section">2024</h2>
<ul>
<li><p>Crowell HL, Leonardo SXM, <strong>Soneson C</strong>, Robinson MD: Author Correction: The shaky foundations of simulating single-cell RNA sequencing data. Genome Biol 25(1):178 (2024). <a href="https://doi.org/10.1186/s13059-024-03329-0" target="_blank"> DOI: 10.1186/s13059-024-03329-0</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/38970115" target="_blank"> PMID 38970115</a>.</p></li>
<li><p>Tiberi S, Meili J, Cai P, <strong>Soneson C</strong>, He D, Sarkar H, Avalos-Pacheco A, Patro R, Robinson MD: DifferentialRegulation: a Bayesian hierarchical approach to identify differentially regulated genes. Biostatistics : (2024). <a href="https://doi.org/10.1093/biostatistics/kxae017" target="_blank"> DOI: 10.1093/biostatistics/kxae017</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/38887902" target="_blank"> PMID 38887902</a>. <a href="https://bioconductor.org/packages/DifferentialRegulation/" target="_blank"> Software/Resources</a>. <a href="https://doi.org/10.1101/2023.08.17.553679" target="_blank"> Preprint</a>.</p></li>
<li><p>Hutchison WJ, Keyes TJ, tidyomics Consortium, Crowell HL, Serizay J, <strong>Soneson C</strong>, Davis ES, Sato N, Moses L, Tarlinton B, Nahid AA, Kosmac M, Clayssen Q, Yuan V, Mu W, Park JE, Mamede I, Ryu MH, Axisa PP, Paiz P, Poon CL, Tang M, Gottardo R, Morgan M, Lee S, Lawrence M, Hicks SC, Nolan GP, Davis KL, Papenfuss AT, Love MI, Mangiola S: The tidyomics ecosystem: enhancing omic data analyses. Nat Methods 21(7):1166-1170 (2024). <a href="https://doi.org/10.1038/s41592-024-02299-2" target="_blank"> DOI: 10.1038/s41592-024-02299-2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/38877315" target="_blank"> PMID 38877315</a>. <a href="https://doi.org/10.1101/2023.09.10.557072" target="_blank"> Preprint</a>.</p></li>
</ul>
</section>
<section id="section-1" class="level2">
<h2 class="anchored" data-anchor-id="section-1">2023</h2>
<ul>
<li><p>Welte T, Goulois A, Stadler MB, Hess D, <strong>Soneson C</strong>, Neagu A, Azzi C, Wisser MJ, Seebacher J, Schmidt I, Estoppey D, Nigsch F, Reece-Hoyes J, Hoepfner D, Großhans H: Convergence of multiple RNA-silencing pathways on GW182/TNRC6. Mol Cell 83(14):2478-2492.e8 (2023). <a href="https://doi.org/10.1016/j.molcel.2023.06.001" target="_blank"> DOI: 10.1016/j.molcel.2023.06.001</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37369201" target="_blank"> PMID 37369201</a>. <a href="https://doi.org/10.1101/2023.04.20.537632" target="_blank"> Preprint</a>.</p></li>
<li><p><strong>Soneson C</strong>, Bendel AM, Diss G, Stadler MB: mutscan-a flexible R package for efficient end-to-end analysis of multiplexed assays of variant effect data. Genome Biol 24(1):132 (2023). <a href="https://doi.org/10.1186/s13059-023-02967-0" target="_blank"> DOI: 10.1186/s13059-023-02967-0</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37264470" target="_blank"> PMID 37264470</a>. <a href="https://github.com/fmicompbio/mutscan_manuscript_2022" target="_blank"> GitHub repo</a>. <a href="https://github.com/fmicompbio/mutscan" target="_blank"> Software/Resources</a>. <a href="https://doi.org/10.1101/2022.10.25.513674" target="_blank"> Preprint</a>.</p></li>
<li><p>Lütge M, De Martin A, Gil-Cruz C, Perez-Shibayama C, Stanossek Y, Onder L, Cheng HW, Kurz L, Cadosch N, <strong>Soneson C</strong>, Robinson MD, Stoeckli SJ, Ludewig B, Pikor NB: Conserved stromal-immune cell circuits secure B cell homeostasis and function. Nat Immunol 24(7):1149-1160 (2023). <a href="https://doi.org/10.1038/s41590-023-01503-3" target="_blank"> DOI: 10.1038/s41590-023-01503-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37202489" target="_blank"> PMID 37202489</a>.</p></li>
<li><p>Sonrel A, Luetge A, <strong>Soneson C</strong>, Mallona I, Germain PL, Knyazev S, Gilis J, Gerber R, Seurinck R, Paul D, Sonder E, Crowell HL, Fanaswala I, Al-Ajami A, Heidari E, Schmeing S, Milosavljevic S, Saeys Y, Mangul S, Robinson MD: Meta-analysis of (single-cell method) benchmarks reveals the need for extensibility and interoperability. Genome Biol 24(1):119 (2023). <a href="https://doi.org/10.1186/s13059-023-02962-5" target="_blank"> DOI: 10.1186/s13059-023-02962-5</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37198712" target="_blank"> PMID 37198712</a>. <a href="https://doi.org/10.1101/2022.09.22.508982" target="_blank"> Preprint</a>.</p></li>
<li><p>Crowell HL, Morillo Leonardo SX, <strong>Soneson C</strong>, Robinson MD: The shaky foundations of simulating single-cell RNA sequencing data. Genome Biol 24(1):62 (2023). <a href="https://doi.org/10.1186/s13059-023-02904-1" target="_blank"> DOI: 10.1186/s13059-023-02904-1</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36991470" target="_blank"> PMID 36991470</a>. <a href="https://doi.org/10.1101/2021.11.15.468676" target="_blank"> Preprint</a>.</p></li>
<li><p>Bastide P, <strong>Soneson C</strong>, Stern DB, Lespinet O, Gallopin M: A Phylogenetic Framework to Simulate Synthetic Interspecies RNA-Seq Data. Mol Biol Evol 40(1): (2023). <a href="https://doi.org/10.1093/molbev/msac269" target="_blank"> DOI: 10.1093/molbev/msac269</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36508357" target="_blank"> PMID 36508357</a>. <a href="https://github.com/i2bc/InterspeciesDE" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/compcodeR/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/2022.01.21.476612v1" target="_blank"> Preprint</a>.</p></li>
<li><p><strong>Soneson C</strong>, Iesmantavicius V, Hess D, Stadler MB, Seebacher J: einprot: flexible, easy-to-use, reproducible workflows for statistical analysis of quantitative proteomics data. The Journal of Open Source Software 8(89):5750 (2023). <a href="https://doi.org/10.21105/joss.05750" target="_blank"> DOI: 10.21105/joss.05750</a>. <a href="https://github.com/fmicompbio/einprot" target="_blank"> Software/Resources</a>. <a href="https://doi.org/10.1101/2023.07.27.550821" target="_blank"> Preprint</a>.</p></li>
</ul>
</section>
<section id="section-2" class="level2">
<h2 class="anchored" data-anchor-id="section-2">2022</h2>
<ul>
<li><p>Prummel KD, Crowell HL, Nieuwenhuize S, Brombacher EC, Daetwyler S, <strong>Soneson C</strong>, Kresoja-Rakic J, Kocere A, Ronner M, Ernst A, Labbaf Z, Clouthier DE, Firulli AB, Sánchez-Iranzo H, Naganathan SR, O’Rourke R, Raz E, Mercader N, Burger A, Felley-Bosco E, Huisken J, Robinson MD, Mosimann C: Hand2 delineates mesothelium progenitors and is reactivated in mesothelioma. Nat Commun 13(1):1677 (2022). <a href="https://doi.org/10.1038/s41467-022-29311-7" target="_blank"> DOI: 10.1038/s41467-022-29311-7</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35354817" target="_blank"> PMID 35354817</a>. <a href="https://www.biorxiv.org/content/10.1101/2020.11.11.355693v3" target="_blank"> Preprint</a>.</p></li>
<li><p>He D, Zakeri M, Sarkar H, <strong>Soneson C</strong>, Srivastava A, Patro R: Alevin-fry unlocks rapid, accurate and memory-frugal quantification of single-cell RNA-seq data. Nat Methods 19(3):316-322 (2022). <a href="https://doi.org/10.1038/s41592-022-01408-3" target="_blank"> DOI: 10.1038/s41592-022-01408-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35277707" target="_blank"> PMID 35277707</a>. <a href="https://www.biorxiv.org/content/10.1101/2021.06.29.450377v2" target="_blank"> Preprint</a>.</p></li>
<li><p>Machlab D, Burger L, <strong>Soneson C</strong>, Rijli FM, Schübeler D, Stadler MB: monaLisa: an R/Bioconductor package for identifying regulatory motifs. Bioinformatics 38(9):2624-2625 (2022). <a href="https://doi.org/10.1093/bioinformatics/btac102" target="_blank"> DOI: 10.1093/bioinformatics/btac102</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35199152" target="_blank"> PMID 35199152</a>. <a href="https://bioconductor.org/packages/monaLisa/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/2021.11.30.470570v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Wagner J, Masek M, Jacobs A, <strong>Soneson C</strong>, Sivapatham S, Damond N, de Souza N, Robinson MD, Bodenmiller B: Mass cytometric and transcriptomic profiling of epithelial-mesenchymal transitions in human mammary cell lines. Sci Data 9(1):44 (2022). <a href="https://doi.org/10.1038/s41597-022-01137-4" target="_blank"> DOI: 10.1038/s41597-022-01137-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35140234" target="_blank"> PMID 35140234</a>. <a href="https://github.com/csoneson/WagnerEMT2020" target="_blank"> GitHub repo</a>. <a href="https://www.biorxiv.org/content/10.1101/2021.03.26.436976v1" target="_blank"> Preprint</a>.</p></li>
</ul>
</section>
<section id="section-3" class="level2">
<h2 class="anchored" data-anchor-id="section-3">2021</h2>
<ul>
<li><p>Huang R, <strong>Soneson C</strong>, Germain PL, Schmidt TSB, Mering CV, Robinson MD: treeclimbR pinpoints the data-dependent resolution of hierarchical hypotheses. Genome Biol 22(1):157 (2021). <a href="https://doi.org/10.1186/s13059-021-02368-1" target="_blank"> DOI: 10.1186/s13059-021-02368-1</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34001188" target="_blank"> PMID 34001188</a>. <a href="https://github.com/fionarhuang/treeclimbR" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/2020.06.08.140608v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Lütge A, Zyprych-Walczak J, Brykczynska Kunzmann U, Crowell HL, Calini D, Malhotra D, <strong>Soneson C</strong>, Robinson MD: CellMixS: quantifying and visualizing batch effects in single-cell RNA-seq data. Life Sci Alliance 4(6): (2021). <a href="https://doi.org/10.26508/lsa.202001004" target="_blank"> DOI: 10.26508/lsa.202001004</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33758076" target="_blank"> PMID 33758076</a>. <a href="https://bioconductor.org/packages/CellMixS/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/2020.12.11.420885v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Kitazawa T, Machlab D, Joshi O, Maiorano N, Kohler H, Ducret S, Kessler S, Gezelius H, <strong>Soneson C</strong>, Papasaikas P, López-Bendito G, Stadler MB, Rijli FM: A unique bipartite Polycomb signature regulates stimulus-response transcription during development. Nat Genet 53(3):379-391 (2021). <a href="https://doi.org/10.1038/s41588-021-00789-z" target="_blank"> DOI: 10.1038/s41588-021-00789-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33603234" target="_blank"> PMID 33603234</a>.</p></li>
<li><p><strong>Soneson C</strong>, Srivastava A, Patro R, Stadler MB: Preprocessing choices affect RNA velocity results for droplet scRNA-seq data. PLoS Comput Biol 17(1):e1008585 (2021). <a href="https://doi.org/10.1371/journal.pcbi.1008585" target="_blank"> DOI: 10.1371/journal.pcbi.1008585</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33428615" target="_blank"> PMID 33428615</a>. <a href="https://github.com/csoneson/rna_velocity_quant" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/eisaR/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990069v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Wang Y, <strong>Soneson C</strong>, Malinowska AL, Laski A, Ghosh S, Kanitz A, Gebert LFR, Robinson MD, Hall J: MiR-CLIP reveals iso-miR selective regulation in the miR-124 targetome. Nucleic Acids Res 49(1):25-37 (2021). <a href="https://doi.org/10.1093/nar/gkaa1117" target="_blank"> DOI: 10.1093/nar/gkaa1117</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33300035" target="_blank"> PMID 33300035</a>.</p></li>
</ul>
</section>
<section id="section-4" class="level2">
<h2 class="anchored" data-anchor-id="section-4">2020</h2>
<ul>
<li><p>Huang R, <strong>Soneson C</strong>, Ernst FGM, Rue-Albrecht KC, Yu G, Hicks SC, Robinson MD: TreeSummarizedExperiment: a S4 class for data with hierarchical structure. F1000Res 9:1246 (2020). <a href="https://doi.org/10.12688/f1000research.26669.2" target="_blank"> DOI: 10.12688/f1000research.26669.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33274053" target="_blank"> PMID 33274053</a>. <a href="https://bioconductor.org/packages/TreeSummarizedExperiment/" target="_blank"> Software/Resources</a>.</p></li>
<li><p>Crowell HL, <strong>Soneson C</strong>, Germain PL, Calini D, Collin L, Raposo C, Malhotra D, Robinson MD: muscat detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data. Nat Commun 11(1):6077 (2020). <a href="https://doi.org/10.1038/s41467-020-19894-4" target="_blank"> DOI: 10.1038/s41467-020-19894-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33257685" target="_blank"> PMID 33257685</a>. <a href="https://bioconductor.org/packages/muscat/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/713412v3" target="_blank"> Preprint</a>.</p></li>
<li><p><strong>Soneson C</strong>, Marini F, Geier F, Love MI, Stadler MB: ExploreModelMatrix: Interactive exploration for improved understanding of design matrices and linear models in R. F1000Res 9:512 (2020). <a href="https://doi.org/10.12688/f1000research.24187.2" target="_blank"> DOI: 10.12688/f1000research.24187.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32704355" target="_blank"> PMID 32704355</a>. <a href="https://bioconductor.org/packages/ExploreModelMatrix/" target="_blank"> Software/Resources</a>.</p></li>
<li><p>Srivastava A, Malik L, Sarkar H, Zakeri M, Almodaresi F, <strong>Soneson C</strong>, Love MI, Kingsford C, Patro R: Alignment and mapping methodology influence transcript abundance estimation. Genome Biol 21(1):239 (2020). <a href="https://doi.org/10.1186/s13059-020-02151-8" target="_blank"> DOI: 10.1186/s13059-020-02151-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32894187" target="_blank"> PMID 32894187</a>. <a href="https://www.biorxiv.org/content/10.1101/657874v2" target="_blank"> Preprint</a>.</p></li>
<li><p>Love MI, <strong>Soneson C</strong>, Hickey PF, Johnson LK, Pierce NT, Shepherd L, Morgan M, Patro R: Tximeta: Reference sequence checksums for provenance identification in RNA-seq. PLoS Comput Biol 16(2):e1007664 (2020). <a href="https://doi.org/10.1371/journal.pcbi.1007664" target="_blank"> DOI: 10.1371/journal.pcbi.1007664</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32097405" target="_blank"> PMID 32097405</a>. <a href="https://bioconductor.org/packages/tximeta/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/777888v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Amezquita RA, Lun ATL, Becht E, Carey VJ, Carpp LN, Geistlinger L, Marini F, Rue-Albrecht K, Risso D, <strong>Soneson C</strong>, Waldron L, Pagès H, Smith ML, Huber W, Morgan M, Gottardo R, Hicks SC: Publisher Correction: Orchestrating single-cell analysis with Bioconductor. Nat Methods 17(2):242 (2020). <a href="https://doi.org/10.1038/s41592-019-0700-8" target="_blank"> DOI: 10.1038/s41592-019-0700-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31827272" target="_blank"> PMID 31827272</a>.</p></li>
<li><p>Amezquita RA, Lun ATL, Becht E, Carey VJ, Carpp LN, Geistlinger L, Marini F, Rue-Albrecht K, Risso D, <strong>Soneson C</strong>, Waldron L, Pagès H, Smith ML, Huber W, Morgan M, Gottardo R, Hicks SC: Orchestrating single-cell analysis with Bioconductor. Nat Methods 17(2):137-145 (2020). <a href="https://doi.org/10.1038/s41592-019-0654-x" target="_blank"> DOI: 10.1038/s41592-019-0654-x</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31792435" target="_blank"> PMID 31792435</a>. <a href="https://osca.bioconductor.org/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/590562v1" target="_blank"> Preprint</a>.</p></li>
</ul>
</section>
<section id="section-5" class="level2">
<h2 class="anchored" data-anchor-id="section-5">2019</h2>
<ul>
<li><p>Weber LM, <strong>Soneson C</strong>: HDCytoData: Collection of high-dimensional cytometry benchmark datasets in Bioconductor object formats. F1000Res 8:1459 (2019). <a href="https://doi.org/10.12688/f1000research.20210.2" target="_blank"> DOI: 10.12688/f1000research.20210.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31857895" target="_blank"> PMID 31857895</a>. <a href="https://bioconductor.org/packages/HDCytoData/" target="_blank"> Software/Resources</a>.</p></li>
<li><p><strong>Soneson C</strong>, Yao Y, Bratus-Neuenschwander A, Patrignani A, Robinson MD, Hussain S: A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes. Nat Commun 10(1):3359 (2019). <a href="https://doi.org/10.1038/s41467-019-11272-z" target="_blank"> DOI: 10.1038/s41467-019-11272-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31366910" target="_blank"> PMID 31366910</a>. <a href="https://github.com/csoneson/NativeRNAseqComplexTranscriptome" target="_blank"> GitHub repo</a>. <a href="https://www.biorxiv.org/content/10.1101/574525v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Weber LM, Saelens W, Cannoodt R, <strong>Soneson C</strong>, Hapfelmeier A, Gardner PP, Boulesteix AL, Saeys Y, Robinson MD: Essential guidelines for computational method benchmarking. Genome Biol 20(1):125 (2019). <a href="https://doi.org/10.1186/s13059-019-1738-8" target="_blank"> DOI: 10.1186/s13059-019-1738-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31221194" target="_blank"> PMID 31221194</a>. <a href="https://arxiv.org/abs/1812.00661" target="_blank"> Preprint</a>.</p></li>
<li><p>Weber LM, Nowicka M, <strong>Soneson C</strong>, Robinson MD: diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering. Commun Biol 2:183 (2019). <a href="https://doi.org/10.1038/s42003-019-0415-5" target="_blank"> DOI: 10.1038/s42003-019-0415-5</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31098416" target="_blank"> PMID 31098416</a>. <a href="https://www.biorxiv.org/content/10.1101/349738v3" target="_blank"> Preprint</a>.</p></li>
<li><p>Orjuela S, Huang R, Hembach KM, Robinson MD, <strong>Soneson C</strong>: ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data. G3 (Bethesda) 9(7):2089-2096 (2019). <a href="https://doi.org/10.1534/g3.119.400185" target="_blank"> DOI: 10.1534/g3.119.400185</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31088905" target="_blank"> PMID 31088905</a>. <a href="https://www.biorxiv.org/content/10.1101/575951v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Cheng HW, Onder L, Novkovic M, <strong>Soneson C</strong>, Lütge M, Pikor N, Scandella E, Robinson MD, Miyazaki JI, Tersteegen A, Sorg U, Pfeffer K, Rülicke T, Hehlgans T, Ludewig B: Origin and differentiation trajectories of fibroblastic reticular cells in the splenic white pulp. Nat Commun 10(1):1739 (2019). <a href="https://doi.org/10.1038/s41467-019-09728-3" target="_blank"> DOI: 10.1038/s41467-019-09728-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30988302" target="_blank"> PMID 30988302</a>.</p></li>
<li><p><strong>Soneson C</strong>, Love MI, Patro R, Hussain S, Malhotra D, Robinson MD: A junction coverage compatibility score to quantify the reliability of transcript abundance estimates and annotation catalogs. Life Sci Alliance 2(1): (2019). <a href="https://doi.org/10.26508/lsa.201800175" target="_blank"> DOI: 10.26508/lsa.201800175</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30655364" target="_blank"> PMID 30655364</a>. <a href="https://github.com/csoneson/annotation_problem_txabundance" target="_blank"> GitHub repo</a>. <a href="https://github.com/csoneson/jcc" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/378539v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Van den Berge K, Hembach KM, <strong>Soneson C</strong>, Tiberi S, Clement L, Love MI, Patro R, Robinson MD: RNA Sequencing Data: Hitchhiker’s Guide to Expression Analysis. Annual Review of Biomedical Data Science 2:139-173 (2019). <a href="https://doi.org/10.1146/annurev-biodatasci-072018-021255" target="_blank"> DOI: 10.1146/annurev-biodatasci-072018-021255</a>.</p></li>
</ul>
</section>
<section id="section-6" class="level2">
<h2 class="anchored" data-anchor-id="section-6">2018</h2>
<ul>
<li><p>Love MI, <strong>Soneson C</strong>, Patro R: Swimming downstream: statistical analysis of differential transcript usage following Salmon quantification. F1000Res 7:952 (2018). <a href="https://doi.org/10.12688/f1000research.15398.3" target="_blank"> DOI: 10.12688/f1000research.15398.3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30356428" target="_blank"> PMID 30356428</a>. <a href="https://github.com/mikelove/swimdown" target="_blank"> GitHub repo</a>.</p></li>
<li><p>Perez-Shibayama C, Gil-Cruz C, Cheng HW, Onder L, Printz A, Mörbe U, Novkovic M, Li C, Lopez-Macias C, Buechler MB, Turley SJ, Mack M, <strong>Soneson C</strong>, Robinson MD, Scandella E, Gommerman J, Ludewig B: Fibroblastic reticular cells initiate immune responses in visceral adipose tissues and secure peritoneal immunity. Sci Immunol 3(26): (2018). <a href="https://doi.org/10.1126/sciimmunol.aar4539" target="_blank"> DOI: 10.1126/sciimmunol.aar4539</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30097537" target="_blank"> PMID 30097537</a>.</p></li>
<li><p>Duò A, Robinson MD, <strong>Soneson C</strong>: A systematic performance evaluation of clustering methods for single-cell RNA-seq data. F1000Res 7:1141 (2018). <a href="https://doi.org/10.12688/f1000research.15666.3" target="_blank"> DOI: 10.12688/f1000research.15666.3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30271584" target="_blank"> PMID 30271584</a>. <a href="https://github.com/markrobinsonuzh/scrnaseq_clustering_comparison" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/DuoClustering2018/" target="_blank"> Software/Resources</a>.</p></li>
<li><p>Rue-Albrecht K, Marini F, <strong>Soneson C</strong>, Lun ATL: iSEE: Interactive SummarizedExperiment Explorer. F1000Res 7:741 (2018). <a href="https://doi.org/10.12688/f1000research.14966.1" target="_blank"> DOI: 10.12688/f1000research.14966.1</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30002819" target="_blank"> PMID 30002819</a>. <a href="https://github.com/ltla/isee2018" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/iSEE/" target="_blank"> Software/Resources</a>.</p></li>
<li><p><strong>Soneson C</strong>, Robinson MD: Bias, robustness and scalability in single-cell differential expression analysis. Nat Methods 15(4):255-261 (2018). <a href="https://doi.org/10.1038/nmeth.4612" target="_blank"> DOI: 10.1038/nmeth.4612</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29481549" target="_blank"> PMID 29481549</a>. <a href="https://github.com/csoneson/conquer_comparison" target="_blank"> GitHub repo</a>. <a href="http://imlspenticton.uzh.ch:3838/conquer/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/143289v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Van den Berge K, Perraudeau F, <strong>Soneson C</strong>, Love MI, Risso D, Vert JP, Robinson MD, Dudoit S, Clement L: Observation weights unlock bulk RNA-seq tools for zero inflation and single-cell applications. Genome Biol 19(1):24 (2018). <a href="https://doi.org/10.1186/s13059-018-1406-4" target="_blank"> DOI: 10.1186/s13059-018-1406-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29478411" target="_blank"> PMID 29478411</a>. <a href="https://github.com/statOmics/zinbwaveZinger" target="_blank"> GitHub repo</a>. <a href="https://www.biorxiv.org/content/10.1101/250126v1" target="_blank"> Preprint</a>.</p></li>
<li><p><strong>Soneson C</strong>, Robinson MD: Towards unified quality verification of synthetic count data with countsimQC. Bioinformatics 34(4):691-692 (2018). <a href="https://doi.org/10.1093/bioinformatics/btx631" target="_blank"> DOI: 10.1093/bioinformatics/btx631</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29028961" target="_blank"> PMID 29028961</a>. <a href="https://github.com/csoneson/countsimQC" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/countsimQC/" target="_blank"> Software/Resources</a>.</p></li>
</ul>
</section>
<section id="section-7" class="level2">
<h2 class="anchored" data-anchor-id="section-7">2017</h2>
<ul>
<li><p>Van den Berge K, <strong>Soneson C</strong>, Robinson MD, Clement L: stageR: a general stage-wise method for controlling the gene-level false discovery rate in differential expression and differential transcript usage. Genome Biol 18(1):151 (2017). <a href="https://doi.org/10.1186/s13059-017-1277-0" target="_blank"> DOI: 10.1186/s13059-017-1277-0</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28784146" target="_blank"> PMID 28784146</a>. <a href="https://bioconductor.org/packages/stageR/" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/109082v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Gille D, Zangger N, <strong>Soneson C</strong>, Bütikofer U, Delorenzi M, Schwander F, Kopf-Bolanz KA, Chollet M, Walther B, Laederach K, Vergères G: Caloric dose-responsive genes in blood cells differentiate the metabolic status of obese men. J Nutr Biochem 43:156-165 (2017). <a href="https://doi.org/10.1016/j.jnutbio.2017.02.012" target="_blank"> DOI: 10.1016/j.jnutbio.2017.02.012</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28319853" target="_blank"> PMID 28319853</a>.</p></li>
<li><p>Neubert NJ, Tillé L, Barras D, <strong>Soneson C</strong>, Baumgaertner P, Rimoldi D, Gfeller D, Delorenzi M, Fuertes Marraco SA, Speiser DE: Broad and Conserved Immune Regulation by Genetically Heterogeneous Melanoma Cells. Cancer Res 77(7):1623-1636 (2017). <a href="https://doi.org/10.1158/0008-5472.CAN-16-2680" target="_blank"> DOI: 10.1158/0008-5472.CAN-16-2680</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28104684" target="_blank"> PMID 28104684</a>.</p></li>
<li><p>Sauerwald J, <strong>Soneson C</strong>, Robinson MD, Luschnig S: Faithful mRNA splicing depends on the Prp19 complex subunit faint sausage and is required for tracheal branching morphogenesis in Drosophila. Development 144(4):657-663 (2017). <a href="https://doi.org/10.1242/dev.144535" target="_blank"> DOI: 10.1242/dev.144535</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087625" target="_blank"> PMID 28087625</a>.</p></li>
<li><p>Zanini IM, <strong>Soneson C</strong>, Lorenzi LE, Azzalin CM: Human cactin interacts with DHX8 and SRRM2 to assure efficient pre-mRNA splicing and sister chromatid cohesion. J Cell Sci 130(4):767-778 (2017). <a href="https://doi.org/10.1242/jcs.194068" target="_blank"> DOI: 10.1242/jcs.194068</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28062851" target="_blank"> PMID 28062851</a>.</p></li>
<li><p>Mathias A, Perriard G, Canales M, <strong>Soneson C</strong>, Delorenzi M, Schluep M, Du Pasquier RA: Increased ex vivo antigen presentation profile of B cells in multiple sclerosis. Mult Scler 23(6):802-809 (2017). <a href="https://doi.org/10.1177/1352458516664210" target="_blank"> DOI: 10.1177/1352458516664210</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27503907" target="_blank"> PMID 27503907</a>.</p></li>
</ul>
</section>
<section id="section-8" class="level2">
<h2 class="anchored" data-anchor-id="section-8">2016</h2>
<ul>
<li><p>Neubert NJ, <strong>Soneson C</strong>, Barras D, Baumgaertner P, Rimoldi D, Delorenzi M, Fuertes Marraco SA, Speiser DE: A Well-Controlled Experimental System to Study Interactions of Cytotoxic T Lymphocytes with Tumor Cells. Front Immunol 7:326 (2016). <a href="https://doi.org/10.3389/fimmu.2016.00326" target="_blank"> DOI: 10.3389/fimmu.2016.00326</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27625650" target="_blank"> PMID 27625650</a>.</p></li>
<li><p><strong>Soneson C</strong>, Robinson MD: iCOBRA: open, reproducible, standardized and live method benchmarking. Nat Methods 13(4):283 (2016). <a href="https://doi.org/10.1038/nmeth.3805" target="_blank"> DOI: 10.1038/nmeth.3805</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27027585" target="_blank"> PMID 27027585</a>. <a href="http://bioconductor.org/packages/release/bioc/html/iCOBRA.html" target="_blank"> Software/Resources</a>. <a href="https://www.biorxiv.org/content/10.1101/033431v1" target="_blank"> Preprint</a>.</p></li>
<li><p>Scholz G, Jandus C, Zhang L, Grandclément C, Lopez-Mejia IC, <strong>Soneson C</strong>, Delorenzi M, Fajas L, Held W, Dormond O, Romero P: Modulation of mTOR Signalling Triggers the Formation of Stem Cell-like Memory T Cells. EBioMedicine 4:50-61 (2016). <a href="https://doi.org/10.1016/j.ebiom.2016.01.019" target="_blank"> DOI: 10.1016/j.ebiom.2016.01.019</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26981571" target="_blank"> PMID 26981571</a>.</p></li>
<li><p><strong>Soneson C</strong>, Matthes KL, Nowicka M, Law CW, Robinson MD: Isoform prefiltering improves performance of count-based methods for analysis of differential transcript usage. Genome Biol 17:12 (2016). <a href="https://doi.org/10.1186/s13059-015-0862-3" target="_blank"> DOI: 10.1186/s13059-015-0862-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813113" target="_blank"> PMID 26813113</a>. <a href="https://github.com/markrobinsonuzh/diff_splice_paper" target="_blank"> GitHub repo</a>. <a href="https://www.biorxiv.org/content/10.1101/025387v2" target="_blank"> Preprint</a>.</p></li>
<li><p>Kuhle J, Barro C, Disanto G, Mathias A, <strong>Soneson C</strong>, Bonnier G, Yaldizli Ö, Regeniter A, Derfuss T, Canales M, Schluep M, Du Pasquier R, Krueger G, Granziera C: Serum neurofilament light chain in early relapsing remitting MS is increased and correlates with CSF levels and with MRI measures of disease severity. Mult Scler 22(12):1550-1559 (2016). <a href="https://doi.org/10.1177/1352458515623365" target="_blank"> DOI: 10.1177/1352458515623365</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26754800" target="_blank"> PMID 26754800</a>.</p></li>
<li><p>Schell MJ, Yang M, Missiaglia E, Delorenzi M, <strong>Soneson C</strong>, Yue B, Nebozhyn MV, Loboda A, Bloom G, Yeatman TJ: A Composite Gene Expression Signature Optimizes Prediction of Colorectal Cancer Metastasis and Outcome. Clin Cancer Res 22(3):734-45 (2016). <a href="https://doi.org/10.1158/1078-0432.CCR-15-0143" target="_blank"> DOI: 10.1158/1078-0432.CCR-15-0143</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26446941" target="_blank"> PMID 26446941</a>.</p></li>
<li><p>Schiffthaler B, Kostadima M, NGS Trainer Consortium, Delhomme N, Rustici G: Training in High-Throughput Sequencing: Common Guidelines to Enable Material Sharing, Dissemination, and Reusability. PLoS Comput Biol 12(6):e1004937 (2016). <a href="https://doi.org/10.1371/journal.pcbi.1004937" target="_blank"> DOI: 10.1371/journal.pcbi.1004937</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27309738" target="_blank"> PMID 27309738</a>.</p></li>
<li><p>SIB Swiss Institute of Bioinformatics Members: The SIB Swiss Institute of Bioinformatics’ resources: focus on curated databases. Nucleic Acids Res 44(D1):D27-37 (2016). <a href="https://doi.org/10.1093/nar/gkv1310" target="_blank"> DOI: 10.1093/nar/gkv1310</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615188" target="_blank"> PMID 26615188</a>.</p></li>
</ul>
</section>
<section id="section-9" class="level2">
<h2 class="anchored" data-anchor-id="section-9">2015</h2>
<ul>
<li><p><strong>Soneson C</strong>, Love MI, Robinson MD: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Res 4:1521 (2015). <a href="https://doi.org/10.12688/f1000research.7563.2" target="_blank"> DOI: 10.12688/f1000research.7563.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26925227" target="_blank"> PMID 26925227</a>. <a href="https://bioconductor.org/packages/tximport/" target="_blank"> Software/Resources</a>.</p></li>
<li><p>Fuertes Marraco SA, <strong>Soneson C</strong>, Delorenzi M, Speiser DE: Genome-wide RNA profiling of long-lasting stem cell-like memory CD8 T cells induced by Yellow Fever vaccination in humans. Genom Data 5:297-301 (2015). <a href="https://doi.org/10.1016/j.gdata.2015.06.024" target="_blank"> DOI: 10.1016/j.gdata.2015.06.024</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26484272" target="_blank"> PMID 26484272</a>.</p></li>
<li><p>Guinney J, Dienstmann R, Wang X, de Reyniès A, Schlicker A, <strong>Soneson C</strong>, Marisa L, Roepman P, Nyamundanda G, Angelino P, Bot BM, Morris JS, Simon IM, Gerster S, Fessler E, De Sousa E Melo F, Missiaglia E, Ramay H, Barras D, Homicsko K, Maru D, Manyam GC, Broom B, Boige V, Perez-Villamil B, Laderas T, Salazar R, Gray JW, Hanahan D, Tabernero J, Bernards R, Friend SH, Laurent-Puig P, Medema JP, Sadanandam A, Wessels L, Delorenzi M, Kopetz S, Vermeulen L, Tejpar S: The consensus molecular subtypes of colorectal cancer. Nat Med 21(11):1350-6 (2015). <a href="https://doi.org/10.1038/nm.3967" target="_blank"> DOI: 10.1038/nm.3967</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26457759" target="_blank"> PMID 26457759</a>.</p></li>
<li><p>Johnsson K, <strong>Soneson C</strong>, Fontes M: Low Bias Local Intrinsic Dimension Estimation from Expected Simplex Skewness. IEEE Trans Pattern Anal Mach Intell 37(1):196-202 (2015). <a href="https://doi.org/10.1109/TPAMI.2014.2343220" target="_blank"> DOI: 10.1109/TPAMI.2014.2343220</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26353219" target="_blank"> PMID 26353219</a>.</p></li>
<li><p>Shchepachev V, Wischnewski H, <strong>Soneson C</strong>, Arnold AW, Azzalin CM: Human Mpn1 promotes post-transcriptional processing and stability of U6atac. FEBS Lett 589(18):2417-23 (2015). <a href="https://doi.org/10.1016/j.febslet.2015.06.046" target="_blank"> DOI: 10.1016/j.febslet.2015.06.046</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26213367" target="_blank"> PMID 26213367</a>.</p></li>
<li><p>Fuertes Marraco SA, <strong>Soneson C</strong>, Cagnon L, Gannon PO, Allard M, Abed Maillard S, Montandon N, Rufer N, Waldvogel S, Delorenzi M, Speiser DE: Long-lasting stem cell-like memory CD8+ T cells with a naïve-like profile upon yellow fever vaccination. Sci Transl Med 7(282):282ra48 (2015). <a href="https://doi.org/10.1126/scitranslmed.aaa3700" target="_blank"> DOI: 10.1126/scitranslmed.aaa3700</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25855494" target="_blank"> PMID 25855494</a>.</p></li>
<li><p>Lorenzi LE, Bah A, Wischnewski H, Shchepachev V, <strong>Soneson C</strong>, Santagostino M, Azzalin CM: Fission yeast Cactin restricts telomere transcription and elongation by controlling Rap1 levels. EMBO J 34(1):115-29 (2015). <a href="https://doi.org/10.15252/embj.201489559" target="_blank"> DOI: 10.15252/embj.201489559</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25398909" target="_blank"> PMID 25398909</a>.</p></li>
</ul>
</section>
<section id="section-10" class="level2">
<h2 class="anchored" data-anchor-id="section-10">2014</h2>
<ul>
<li><p><strong>Soneson C</strong>, Gerster S, Delorenzi M: Batch effect confounding leads to strong bias in performance estimates obtained by cross-validation. PLoS One 9(6):e100335 (2014). <a href="https://doi.org/10.1371/journal.pone.0100335" target="_blank"> DOI: 10.1371/journal.pone.0100335</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24967636" target="_blank"> PMID 24967636</a>.</p></li>
<li><p><strong>Soneson C</strong>: compcodeR–an R package for benchmarking differential expression methods for RNA-seq data. Bioinformatics 30(17):2517-8 (2014). <a href="https://doi.org/10.1093/bioinformatics/btu324" target="_blank"> DOI: 10.1093/bioinformatics/btu324</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24813215" target="_blank"> PMID 24813215</a>. <a href="https://bioconductor.org/packages/compcodeR/" target="_blank"> Software/Resources</a>.</p></li>
<li><p>Sun Y, Zhang W, Chen D, Lv Y, Zheng J, Lilljebjörn H, Ran L, Bao Z, <strong>Soneson C</strong>, Sjögren HO, Salford LG, Ji J, French PJ, Fioretos T, Jiang T, Fan X: A glioma classification scheme based on coexpression modules of EGFR and PDGFRA. Proc Natl Acad Sci U S A 111(9):3538-43 (2014). <a href="https://doi.org/10.1073/pnas.1313814111" target="_blank"> DOI: 10.1073/pnas.1313814111</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24550449" target="_blank"> PMID 24550449</a>.</p></li>
<li><p>Duffy D, Mamdouh R, Laird M, <strong>Soneson C</strong>, Le Fouler L, El-Daly M, Casrouge A, Decalf J, Abbas A, Eldin NS, Fontes M, Abdel-Hamid M, Mohamed MK, Rafik M, Fontanet A, Albert ML: The ABCs of viral hepatitis that define biomarker signatures of acute viral hepatitis. Hepatology 59(4):1273-82 (2014). <a href="https://doi.org/10.1002/hep.26901" target="_blank"> DOI: 10.1002/hep.26901</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24500966" target="_blank"> PMID 24500966</a>.</p></li>
<li><p>Missiaglia E, Jacobs B, D’Ario G, Di Narzo AF, <strong>Soneson C</strong>, Budinska E, Popovici V, Vecchione L, Gerster S, Yan P, Roth AD, Klingbiel D, Bosman FT, Delorenzi M, Tejpar S: Distal and proximal colon cancers differ in terms of molecular, pathological, and clinical features. Ann Oncol 25(10):1995-2001 (2014). <a href="https://doi.org/10.1093/annonc/mdu275" target="_blank"> DOI: 10.1093/annonc/mdu275</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25057166" target="_blank"> PMID 25057166</a>.</p></li>
<li><p><strong>Soneson C</strong>, Fontes M: Incorporation of gene exchangeabilities improves the reproducibility of gene set rankings. Computational Statistics &amp; Data Analysis 71:588-598 (2014). <a href="https://doi.org/10.1016/j.csda.2012.07.026" target="_blank"> DOI: 10.1016/j.csda.2012.07.026</a>.</p></li>
</ul>
</section>
<section id="section-11" class="level2">
<h2 class="anchored" data-anchor-id="section-11">2013</h2>
<ul>
<li><p><strong>Soneson C</strong>, Delorenzi M: A comparison of methods for differential expression analysis of RNA-seq data. BMC Bioinformatics 14:91 (2013). <a href="https://doi.org/10.1186/1471-2105-14-91" target="_blank"> DOI: 10.1186/1471-2105-14-91</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23497356" target="_blank"> PMID 23497356</a>.</p></li>
<li><p><strong>Soneson C</strong>, Torstensson A: Manliga och kvinnliga teknologers självbilder och deras stereotypbilder av teknologer. Högre Utbildning 3(1):3-14 (2013). .</p></li>
</ul>
</section>
<section id="section-12" class="level2">
<h2 class="anchored" data-anchor-id="section-12">2012</h2>
<ul>
<li><p>Shchepachev V, Wischnewski H, Missiaglia E, <strong>Soneson C</strong>, Azzalin CM: Mpn1, mutated in poikiloderma with neutropenia protein 1, is a conserved 3’-to-5’ RNA exonuclease processing U6 small nuclear RNA. Cell Rep 2(4):855-65 (2012). <a href="https://doi.org/10.1016/j.celrep.2012.08.031" target="_blank"> DOI: 10.1016/j.celrep.2012.08.031</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23022480" target="_blank"> PMID 23022480</a>.</p></li>
<li><p><strong>Soneson C</strong>, Fontes M: A framework for list representation, enabling list stabilization through incorporation of gene exchangeabilities. Biostatistics 13(1):129-41 (2012). <a href="https://doi.org/10.1093/biostatistics/kxr023" target="_blank"> DOI: 10.1093/biostatistics/kxr023</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21908866" target="_blank"> PMID 21908866</a>.</p></li>
</ul>
</section>
<section id="section-13" class="level2">
<h2 class="anchored" data-anchor-id="section-13">2011</h2>
<ul>
<li><p>Fontes M, <strong>Soneson C</strong>: The projection score–an evaluation criterion for variable subset selection in PCA visualization. BMC Bioinformatics 12:307 (2011). <a href="https://doi.org/10.1186/1471-2105-12-307" target="_blank"> DOI: 10.1186/1471-2105-12-307</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21798031" target="_blank"> PMID 21798031</a>.</p></li>
<li><p><strong>Soneson C</strong>, Fontes M: A method for visual identification of small sample subgroups and potential biomarkers. Annals of Applied Statistics 5(3):2131-2149 (2011). <a href="https://doi.org/10.1214/11-AOAS460" target="_blank"> DOI: 10.1214/11-AOAS460</a>.</p></li>
</ul>
</section>
<section id="section-14" class="level2">
<h2 class="anchored" data-anchor-id="section-14">2010</h2>
<ul>
<li><p><strong>Soneson C</strong>, Fontes M, Zhou Y, Denisov V, Paulsen JS, Kirik D, Petersén A, Huntington Study Group PREDICT-HD investigators: Early changes in the hypothalamic region in prodromal Huntington disease revealed by MRI analysis. Neurobiol Dis 40(3):531-43 (2010). <a href="https://doi.org/10.1016/j.nbd.2010.07.013" target="_blank"> DOI: 10.1016/j.nbd.2010.07.013</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20682340" target="_blank"> PMID 20682340</a>.</p></li>
<li><p>Lilljebjörn H, <strong>Soneson C</strong>, Andersson A, Heldrup J, Behrendtz M, Kawamata N, Ogawa S, Koeffler HP, Mitelman F, Johansson B, Fontes M, Fioretos T: The correlation pattern of acquired copy number changes in 164 ETV6/RUNX1-positive childhood acute lymphoblastic leukemias. Hum Mol Genet 19(16):3150-8 (2010). <a href="https://doi.org/10.1093/hmg/ddq224" target="_blank"> DOI: 10.1093/hmg/ddq224</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20513752" target="_blank"> PMID 20513752</a>.</p></li>
<li><p><strong>Soneson C</strong>, Lilljebjörn H, Fioretos T, Fontes M: Integrative analysis of gene expression and copy number alterations using canonical correlation analysis. BMC Bioinformatics 11:191 (2010). <a href="https://doi.org/10.1186/1471-2105-11-191" target="_blank"> DOI: 10.1186/1471-2105-11-191</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20398334" target="_blank"> PMID 20398334</a>.</p></li>
</ul>
</section>
<section id="section-15" class="level2">
<h2 class="anchored" data-anchor-id="section-15">2009</h2>
<ul>
<li>Björklund T, Hall H, Breysse N, <strong>Soneson C</strong>, Carlsson T, Mandel RJ, Carta M, Kirik D: Optimization of continuous in vivo DOPA production and studies on ectopic DA synthesis using rAAV5 vectors in Parkinsonian rats. J Neurochem 111(2):355-67 (2009). <a href="https://doi.org/10.1111/j.1471-4159.2009.06340.x" target="_blank"> DOI: 10.1111/j.1471-4159.2009.06340.x</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19686242" target="_blank"> PMID 19686242</a>.</li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>